
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://monarch-initiative.github.io/namo/elements/">
      
      
        <link rel="prev" href="../how-to/curate-organ-on-chip-paper/">
      
      
        <link rel="next" href="../about/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.16">
    
    
      
        <title>Schema - namo</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e37652d.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../css/ansi-colours.css">
    
      <link rel="stylesheet" href="../css/jupyter-cells.css">
    
      <link rel="stylesheet" href="../css/pandas-dataframe.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    <body dir="ltr">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#namo" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="namo" class="md-header__button md-logo" aria-label="namo" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            namo
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Schema
            
          </span>
        </div>
      </div>
    </div>
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/monarch-initiative/namo" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path fill="currentColor" d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="namo" class="md-nav__button md-logo" aria-label="namo" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    namo
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/monarch-initiative/namo" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path fill="currentColor" d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../use-cases/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Use Cases
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    How-to
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            How-to
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../how-to/curate/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../how-to/curate-organoid-paper/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Curate Organoid Papers
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../how-to/curate-organ-on-chip-paper/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Curate Organ-on-Chip Papers
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    Schema
    
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    Schema
    
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    <span class="md-ellipsis">
      Classes
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    <span class="md-ellipsis">
      Slots
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      Enumerations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    <span class="md-ellipsis">
      Types
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      Subsets
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../about/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    About
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    <span class="md-ellipsis">
      Classes
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    <span class="md-ellipsis">
      Slots
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      Enumerations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    <span class="md-ellipsis">
      Types
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      Subsets
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="namo">namo</h1>
<p>New Approach Methodology Ontology and Schema</p>
<p>URI: https://w3id.org/monarch-initiative/namo</p>
<p>Name: namo</p>
<h2 id="classes">Classes</h2>
<table>
<thead>
<tr>
<th>Class</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="CellRatio/">CellRatio</a></td>
<td>Ratio specification for different cell types in co-culture systems</td>
</tr>
<tr>
<td><a href="CellTypeProportion/">CellTypeProportion</a></td>
<td>Quantitative comparison of cell type proportions between systems</td>
</tr>
<tr>
<td><a href="ChannelDimensions/">ChannelDimensions</a></td>
<td>Dimensions of a microfluidic channel</td>
</tr>
<tr>
<td><a href="ConcordanceResult/">ConcordanceResult</a></td>
<td></td>
</tr>
<tr>
<td><a href="CrossValidation/">CrossValidation</a></td>
<td>Cross-validation strategy and results for ML models</td>
</tr>
<tr>
<td><a href="Dataset/">Dataset</a></td>
<td></td>
</tr>
<tr>
<td><a href="DoseResponseSimilarity/">DoseResponseSimilarity</a></td>
<td>Comparison of dose-response relationships between model and biological system...</td>
</tr>
<tr>
<td><a href="DrugProperties/">DrugProperties</a></td>
<td>Physicochemical and pharmacological properties of a drug in PBPK models</td>
</tr>
<tr>
<td><a href="EnrichmentStatistics/">EnrichmentStatistics</a></td>
<td>Statistical measures for pathway enrichment analysis</td>
</tr>
<tr>
<td><a href="ModelPerformance/">ModelPerformance</a></td>
<td>Statistical performance metrics for computational models</td>
</tr>
<tr>
<td><a href="ModelsRelationship/">ModelsRelationship</a></td>
<td></td>
</tr>
<tr>
<td><a href="NamedThing/">NamedThing</a></td>
<td>A generic grouping for any identifiable entity</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="BiologicalSystem/">BiologicalSystem</a></td>
<td></td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="CellTypeCoverage/">CellTypeCoverage</a></td>
<td>Assessment of cell type representation and cellular diversity between systems</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="FunctionalAssay/">FunctionalAssay</a></td>
<td>A functional assay used to assess biological capabilities</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="FunctionalParity/">FunctionalParity</a></td>
<td>Evaluation of functional capabilities and physiological responses between sys...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Gene/">Gene</a></td>
<td>A gene entity with identifiers and expression information</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="MechanicalStimulation/">MechanicalStimulation</a></td>
<td>Specification of mechanical forces applied to the model system</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="MicrofluidicDesign/">MicrofluidicDesign</a></td>
<td>Detailed specification of a microfluidic device design including its architec...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ModelSystem/">ModelSystem</a></td>
<td></td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="AnimalModel/">AnimalModel</a></td>
<td></td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="NAMModel/">NAMModel</a></td>
<td>A New Approach Methodology (NAM) model, which is a type of model system that ...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="CellularSystem/">CellularSystem</a></td>
<td>Cell-based model systems that use living cells to model biological processes</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="CoCulture/">CoCulture</a></td>
<td>Co-culture systems combining multiple cell types to mimic  microenvironments ...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ThreeDCellCulture/">ThreeDCellCulture</a></td>
<td>Three-dimensional cell culture systems including spheroids and organoids</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Organoid/">Organoid</a></td>
<td>A 3D cell culture system that self-organizes to recapitulate key structural a...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="TwoDCellCulture/">TwoDCellCulture</a></td>
<td>Conventional monolayer cell cultures grown on flat surfaces</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="CellLineModel/">CellLineModel</a></td>
<td>A model system based on immortalized cell lines that can be maintained in cul...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="InSilicoModel/">InSilicoModel</a></td>
<td>Computational models that simulate biological processes without physical biol...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="DigitalTwin/">DigitalTwin</a></td>
<td>Computational replicas of biological systems for real-time prediction and per...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="MetabolicModel/">MetabolicModel</a></td>
<td>A model that simulates the metabolic processes of an organism or system</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="MLModel/">MLModel</a></td>
<td>Machine Learning and AI-based models for prediction, mechanism inference, and...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="PBPKModel/">PBPKModel</a></td>
<td>Physiologically Based Pharmacokinetic models that simulate drug  absorption, ...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="QSARModel/">QSARModel</a></td>
<td>Quantitative Structure-Activity Relationship models that predict  chemical/bi...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="MicrophysiologicalSystem/">MicrophysiologicalSystem</a></td>
<td>Organ-/tissue-on-chip systems that integrate microfluidics, biomaterials,  an...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="OrganOnChip/">OrganOnChip</a></td>
<td>A model system that simulates the physiological functions of an organ using a...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="TissueOnChip/">TissueOnChip</a></td>
<td>Tissue-level microphysiological systems that model specific tissue functions ...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="MolecularSimilarity/">MolecularSimilarity</a></td>
<td>Detailed assessment of molecular-level concordance between model and biologic...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Pathway/">Pathway</a></td>
<td>A biological pathway with activity and enrichment information</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="PathwayConcordance/">PathwayConcordance</a></td>
<td>Assessment of biological pathway conservation and activity between model and ...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="PBPKCompartment/">PBPKCompartment</a></td>
<td>A physiological compartment in a PBPK model</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="PhenotypeOverlap/">PhenotypeOverlap</a></td>
<td>Comparison of phenotypic manifestations between model and biological systems</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Reproducibility/">Reproducibility</a></td>
<td>Assessment of experimental reproducibility and consistency of the model syste...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Study/">Study</a></td>
<td>A study is a structured investigation or analysis, often involving the collec...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Term/">Term</a></td>
<td>A term is a concept or entity that can be defined and used in a specific cont...</td>
</tr>
<tr>
<td><a href="QualityControlMetric/">QualityControlMetric</a></td>
<td>A quality control measure and its associated value</td>
</tr>
<tr>
<td><a href="Reference/">Reference</a></td>
<td>A literature reference with identifier and title for citing published work</td>
</tr>
<tr>
<td><a href="StatisticalSignificance/">StatisticalSignificance</a></td>
<td>Statistical measures of significance for molecular comparisons</td>
</tr>
<tr>
<td><a href="StructuredConcordanceResult/">StructuredConcordanceResult</a></td>
<td>Detailed structured assessment of concordance between model and biological sy...</td>
</tr>
</tbody>
</table>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Slot</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="accuracy/">accuracy</a></td>
<td>Overall accuracy of the model (0</td>
</tr>
<tr>
<td><a href="active_pathways/">active_pathways</a></td>
<td>List of biological pathways that are active in both systems</td>
</tr>
<tr>
<td><a href="activity_endpoint/">activity_endpoint</a></td>
<td>Biological activity or property being predicted</td>
</tr>
<tr>
<td><a href="activity_score/">activity_score</a></td>
<td>Quantitative measure of pathway activity</td>
</tr>
<tr>
<td><a href="adjusted_p_value/">adjusted_p_value</a></td>
<td>Multiple testing corrected p-value</td>
</tr>
<tr>
<td><a href="age/">age</a></td>
<td>The age of the animal used in the model system</td>
</tr>
<tr>
<td><a href="architecture_type/">architecture_type</a></td>
<td>The overall architecture type of the microfluidic device</td>
</tr>
<tr>
<td><a href="assay_result/">assay_result</a></td>
<td>Quantitative result of the assay</td>
</tr>
<tr>
<td><a href="assay_type/">assay_type</a></td>
<td>Type of functional assay (e</td>
</tr>
<tr>
<td><a href="auc/">auc</a></td>
<td>Area under the ROC curve</td>
</tr>
<tr>
<td><a href="authentication_method/">authentication_method</a></td>
<td>Method used for cell line authentication (e</td>
</tr>
<tr>
<td><a href="authors/">authors</a></td>
<td>Authors of the publication</td>
</tr>
<tr>
<td><a href="barrier_functions/">barrier_functions</a></td>
<td>Tissue barrier functions modeled (epithelial, endothelial, etc</td>
</tr>
<tr>
<td><a href="batch_to_batch_variation/">batch_to_batch_variation</a></td>
<td>Measure of variation between different experimental batches</td>
</tr>
<tr>
<td><a href="biological_context/">biological_context</a></td>
<td>tissue/region (anatomy), cell types, sex/age equivalents, mechanics (e</td>
</tr>
<tr>
<td><a href="biological_organization_level/">biological_organization_level</a></td>
<td>The level of biological organization represented by the model</td>
</tr>
<tr>
<td><a href="biological_proportion/">biological_proportion</a></td>
<td>Proportion of this cell type in the biological system</td>
</tr>
<tr>
<td><a href="biological_specific_phenotypes/">biological_specific_phenotypes</a></td>
<td>List of phenotypes present only in the biological system</td>
</tr>
<tr>
<td><a href="biological_system_modeled/">biological_system_modeled</a></td>
<td></td>
</tr>
<tr>
<td><a href="blood_flow/">blood_flow</a></td>
<td>Blood flow to the compartment (L/h)</td>
</tr>
<tr>
<td><a href="cell_ratios/">cell_ratios</a></td>
<td>Ratios of different cell types in the co-culture</td>
</tr>
<tr>
<td><a href="cell_source/">cell_source</a></td>
<td>Source of cells (e</td>
</tr>
<tr>
<td><a href="cell_type/">cell_type</a></td>
<td>The cell type for which the ratio is specified</td>
</tr>
<tr>
<td><a href="cell_type_coverage/">cell_type_coverage</a></td>
<td></td>
</tr>
<tr>
<td><a href="cell_type_proportions/">cell_type_proportions</a></td>
<td>Quantitative comparison of cell type proportions</td>
</tr>
<tr>
<td><a href="cell_types/">cell_types</a></td>
<td>Cell types present in the cellular system</td>
</tr>
<tr>
<td><a href="channel_configuration/">channel_configuration</a></td>
<td>Configuration of channels (e</td>
</tr>
<tr>
<td><a href="channel_dimensions/">channel_dimensions</a></td>
<td>Dimensions of the channels in the device</td>
</tr>
<tr>
<td><a href="channel_name/">channel_name</a></td>
<td>Name or identifier of the channel (e</td>
</tr>
<tr>
<td><a href="clearance/">clearance</a></td>
<td>Total body clearance (L/h)</td>
</tr>
<tr>
<td><a href="coculture_configuration/">coculture_configuration</a></td>
<td>Configuration of co-culture (direct contact, transwell, conditioned media)</td>
</tr>
<tr>
<td><a href="coefficient_of_variation/">coefficient_of_variation</a></td>
<td>Coefficient of variation across experimental replicates</td>
</tr>
<tr>
<td><a href="compartment_type/">compartment_type</a></td>
<td>Type of physiological compartment</td>
</tr>
<tr>
<td><a href="compartments/">compartments</a></td>
<td>Physiological compartments included in the model</td>
</tr>
<tr>
<td><a href="complexity_level/">complexity_level</a></td>
<td>Level of biological complexity represented (subcellular, cellular, tissue, or...</td>
</tr>
<tr>
<td><a href="compound_tested/">compound_tested</a></td>
<td>Name of compound used in dose-response testing</td>
</tr>
<tr>
<td><a href="computational_method/">computational_method</a></td>
<td>Primary computational method or algorithm used</td>
</tr>
<tr>
<td><a href="concordance/">concordance</a></td>
<td>Metrics used to assess the concordance between the model system and the biolo...</td>
</tr>
<tr>
<td><a href="confidence_interval_lower/">confidence_interval_lower</a></td>
<td>Lower bound of confidence interval</td>
</tr>
<tr>
<td><a href="confidence_interval_upper/">confidence_interval_upper</a></td>
<td>Upper bound of confidence interval</td>
</tr>
<tr>
<td><a href="confluence_level/">confluence_level</a></td>
<td>Typical confluence level maintained (0</td>
</tr>
<tr>
<td><a href="conserved_functions/">conserved_functions</a></td>
<td>List of biological functions conserved between model and biological system</td>
</tr>
<tr>
<td><a href="conserved_genes/">conserved_genes</a></td>
<td>List of genes with conserved expression patterns between model and target</td>
</tr>
<tr>
<td><a href="context_of_use/">context_of_use</a></td>
<td>What decision will this inform? Care? Policy? Drug approval?</td>
</tr>
<tr>
<td><a href="correlation_coefficient/">correlation_coefficient</a></td>
<td>Pearson correlation coefficient for expression profiles</td>
</tr>
<tr>
<td><a href="coverage_percentage/">coverage_percentage</a></td>
<td>Percentage of target cell types represented in the model system</td>
</tr>
<tr>
<td><a href="cross_validation/">cross_validation</a></td>
<td>Cross-validation strategy and results</td>
</tr>
<tr>
<td><a href="culture_conditions/">culture_conditions</a></td>
<td>Standard culture conditions and media used</td>
</tr>
<tr>
<td><a href="culture_system/">culture_system</a></td>
<td>Culture system used (e</td>
</tr>
<tr>
<td><a href="cv_method/">cv_method</a></td>
<td>Type of cross-validation used</td>
</tr>
<tr>
<td><a href="cv_score/">cv_score</a></td>
<td>Average cross-validation score</td>
</tr>
<tr>
<td><a href="cv_std/">cv_std</a></td>
<td>Standard deviation of cross-validation scores</td>
</tr>
<tr>
<td><a href="cyclic_stretch_percent/">cyclic_stretch_percent</a></td>
<td>Percentage of cyclic stretch applied (if applicable)</td>
</tr>
<tr>
<td><a href="data_source/">data_source</a></td>
<td>Source of molecular data (e</td>
</tr>
<tr>
<td><a href="description/">description</a></td>
<td>A human-readable description for a thing</td>
</tr>
<tr>
<td><a href="differentially_expressed_genes/">differentially_expressed_genes</a></td>
<td>List of genes that are differentially expressed in the model system</td>
</tr>
<tr>
<td><a href="differentiation_method/">differentiation_method</a></td>
<td>Method used to differentiate cells into organoid (e</td>
</tr>
<tr>
<td><a href="divergent_pathways/">divergent_pathways</a></td>
<td>List of pathways that show different activity patterns</td>
</tr>
<tr>
<td><a href="dose_response_similarity/">dose_response_similarity</a></td>
<td>Comparison of dose-response relationships for therapeutic compounds</td>
</tr>
<tr>
<td><a href="drug_properties/">drug_properties</a></td>
<td>Physicochemical and pharmacological properties modeled</td>
</tr>
<tr>
<td><a href="duration_minutes/">duration_minutes</a></td>
<td>Duration of mechanical stimulation in minutes</td>
</tr>
<tr>
<td><a href="ec50_ratio/">ec50_ratio</a></td>
<td>Ratio of EC50 values between model and biological system</td>
</tr>
<tr>
<td><a href="elimination_pathways/">elimination_pathways</a></td>
<td>Drug elimination and metabolism pathways included</td>
</tr>
<tr>
<td><a href="endpoints/">endpoints</a></td>
<td>phenotypes, function (TEER/leak, beating rate), and multi-omics</td>
</tr>
<tr>
<td><a href="enrichment_score/">enrichment_score</a></td>
<td>Statistical enrichment score for the pathway</td>
</tr>
<tr>
<td><a href="enrichment_statistics/">enrichment_statistics</a></td>
<td>Statistical measures of pathway enrichment</td>
</tr>
<tr>
<td><a href="ensembl_id/">ensembl_id</a></td>
<td>Ensembl gene identifier</td>
</tr>
<tr>
<td><a href="entrez_id/">entrez_id</a></td>
<td>NCBI Entrez gene identifier</td>
</tr>
<tr>
<td><a href="environment/">environment</a></td>
<td>The environmental conditions under which the animal model is maintained</td>
</tr>
<tr>
<td><a href="feature_types/">feature_types</a></td>
<td>Types of features used (molecular, phenotypic, imaging, etc</td>
</tr>
<tr>
<td><a href="flow_control_method/">flow_control_method</a></td>
<td>Methods used to control fluid flow in the device</td>
</tr>
<tr>
<td><a href="fold_change/">fold_change</a></td>
<td>Fold change in expression compared to control or reference</td>
</tr>
<tr>
<td><a href="frequency_hz/">frequency_hz</a></td>
<td>Frequency of mechanical stimulation in Hertz</td>
</tr>
<tr>
<td><a href="functional_assays/">functional_assays</a></td>
<td>List of functional assays used to assess parity</td>
</tr>
<tr>
<td><a href="functional_parity/">functional_parity</a></td>
<td></td>
</tr>
<tr>
<td><a href="functional_similarity_score/">functional_similarity_score</a></td>
<td>Quantitative score (0</td>
</tr>
<tr>
<td><a href="gene_symbol/">gene_symbol</a></td>
<td>Standard gene symbol (e</td>
</tr>
<tr>
<td><a href="genes_in_dataset/">genes_in_dataset</a></td>
<td>Number of genes from dataset found in pathway</td>
</tr>
<tr>
<td><a href="genes_in_pathway/">genes_in_pathway</a></td>
<td>Number of genes in the pathway</td>
</tr>
<tr>
<td><a href="height/">height</a></td>
<td>Height of the channel in micrometers</td>
</tr>
<tr>
<td><a href="id/">id</a></td>
<td>A unique identifier for a thing</td>
</tr>
<tr>
<td><a href="impaired_functions/">impaired_functions</a></td>
<td>List of functions that are impaired or absent in the model system</td>
</tr>
<tr>
<td><a href="inter_laboratory_consistency/">inter_laboratory_consistency</a></td>
<td>Measure of consistency across different laboratories</td>
</tr>
<tr>
<td><a href="interaction_mechanisms/">interaction_mechanisms</a></td>
<td>Mechanisms of cell-cell interaction (paracrine, direct contact, mechanical)</td>
</tr>
<tr>
<td><a href="interface_type/">interface_type</a></td>
<td>Type of interface(s) present in the device</td>
</tr>
<tr>
<td><a href="is_computed/">is_computed</a></td>
<td>Indicates whether the model is computed or derived from experimental data</td>
</tr>
<tr>
<td><a href="journal/">journal</a></td>
<td>Journal or publication venue</td>
</tr>
<tr>
<td><a href="length/">length</a></td>
<td>Length of the channel in millimeters</td>
</tr>
<tr>
<td><a href="logp/">logp</a></td>
<td>Lipophilicity (log P)</td>
</tr>
<tr>
<td><a href="material/">material</a></td>
<td>Materials used to construct the device</td>
</tr>
<tr>
<td><a href="matrix_composition/">matrix_composition</a></td>
<td>Composition of extracellular matrix or scaffold material</td>
</tr>
<tr>
<td><a href="max_response_ratio/">max_response_ratio</a></td>
<td>Ratio of maximum responses between systems</td>
</tr>
<tr>
<td><a href="mechanical_forces/">mechanical_forces</a></td>
<td>Mechanical forces applied to the model system</td>
</tr>
<tr>
<td><a href="membrane_pore_size/">membrane_pore_size</a></td>
<td>Pore size of the membrane in micrometers</td>
</tr>
<tr>
<td><a href="membrane_thickness/">membrane_thickness</a></td>
<td>Thickness of the membrane in micrometers</td>
</tr>
<tr>
<td><a href="membrane_type/">membrane_type</a></td>
<td>Type of membrane used in the device if applicable</td>
</tr>
<tr>
<td><a href="methodology/">methodology</a></td>
<td>Description of experimental methods used for molecular comparison</td>
</tr>
<tr>
<td><a href="metric_name/">metric_name</a></td>
<td>Name of the quality control metric</td>
</tr>
<tr>
<td><a href="metric_value/">metric_value</a></td>
<td>Value of the quality control metric</td>
</tr>
<tr>
<td><a href="microfluidic_design/">microfluidic_design</a></td>
<td>Detailed design specifications of the microfluidic device</td>
</tr>
<tr>
<td><a href="missing_cell_types/">missing_cell_types</a></td>
<td>List of cell types present in biological system but missing in model</td>
</tr>
<tr>
<td><a href="ml_algorithm/">ml_algorithm</a></td>
<td>Type of machine learning algorithm used</td>
</tr>
<tr>
<td><a href="model_interpretability/">model_interpretability</a></td>
<td>Level of model interpretability (black box, interpretable, explainable)</td>
</tr>
<tr>
<td><a href="model_performance/">model_performance</a></td>
<td>Statistical performance metrics of the model</td>
</tr>
<tr>
<td><a href="model_proportion/">model_proportion</a></td>
<td>Proportion of this cell type in the model system</td>
</tr>
<tr>
<td><a href="model_specific_phenotypes/">model_specific_phenotypes</a></td>
<td>List of phenotypes present only in the model system</td>
</tr>
<tr>
<td><a href="model_systems/">model_systems</a></td>
<td></td>
</tr>
<tr>
<td><a href="models/">models</a></td>
<td></td>
</tr>
<tr>
<td><a href="molecular_descriptors/">molecular_descriptors</a></td>
<td>Types of molecular descriptors used (topological, electronic, etc</td>
</tr>
<tr>
<td><a href="molecular_similarity/">molecular_similarity</a></td>
<td></td>
</tr>
<tr>
<td><a href="molecular_weight/">molecular_weight</a></td>
<td>Molecular weight (g/mol)</td>
</tr>
<tr>
<td><a href="n_folds/">n_folds</a></td>
<td>Number of folds in cross-validation</td>
</tr>
<tr>
<td><a href="name/">name</a></td>
<td>A human-readable name for a thing</td>
</tr>
<tr>
<td><a href="number_of_channels/">number_of_channels</a></td>
<td>Total number of channels in the device</td>
</tr>
<tr>
<td><a href="organ_modeled/">organ_modeled</a></td>
<td>The organ or tissue being modeled</td>
</tr>
<tr>
<td><a href="p_value/">p_value</a></td>
<td>Statistical p-value for differential expression</td>
</tr>
<tr>
<td><a href="partition_coefficient/">partition_coefficient</a></td>
<td>Tissue-to-plasma partition coefficient</td>
</tr>
<tr>
<td><a href="pass_fail_status/">pass_fail_status</a></td>
<td>Whether this metric passes quality control criteria</td>
</tr>
<tr>
<td><a href="passage_protocol/">passage_protocol</a></td>
<td>Standard passaging protocol and frequency</td>
</tr>
<tr>
<td><a href="passage_range/">passage_range</a></td>
<td>Recommended passage number range for experimental use</td>
</tr>
<tr>
<td><a href="pathway_analysis_method/">pathway_analysis_method</a></td>
<td>Method used for pathway analysis (e</td>
</tr>
<tr>
<td><a href="pathway_concordance/">pathway_concordance</a></td>
<td></td>
</tr>
<tr>
<td><a href="pathway_database/">pathway_database</a></td>
<td>Source database (e</td>
</tr>
<tr>
<td><a href="pathway_id/">pathway_id</a></td>
<td>Database-specific pathway identifier</td>
</tr>
<tr>
<td><a href="pathway_overlap_score/">pathway_overlap_score</a></td>
<td>Quantitative score (0</td>
</tr>
<tr>
<td><a href="perfusion_system/">perfusion_system</a></td>
<td>Description of perfusion and flow systems</td>
</tr>
<tr>
<td><a href="personalization_parameters/">personalization_parameters</a></td>
<td>Parameters used for personalization (genetic, phenotypic, etc</td>
</tr>
<tr>
<td><a href="perturbations/">perturbations</a></td>
<td>exposure/dose/time; diet/drugs/toxicants</td>
</tr>
<tr>
<td><a href="phenotype_ontology/">phenotype_ontology</a></td>
<td>Ontology used for phenotype classification (e</td>
</tr>
<tr>
<td><a href="phenotype_overlap/">phenotype_overlap</a></td>
<td></td>
</tr>
<tr>
<td><a href="phenotype_similarity_score/">phenotype_similarity_score</a></td>
<td>Quantitative score (0</td>
</tr>
<tr>
<td><a href="pka/">pka</a></td>
<td>Acid dissociation constant</td>
</tr>
<tr>
<td><a href="plan_comparators/">plan_comparators</a></td>
<td>human data, gold-standard assays, or high-quality animal references</td>
</tr>
<tr>
<td><a href="prediction_scope/">prediction_scope</a></td>
<td>Scope and limitations of model predictions</td>
</tr>
<tr>
<td><a href="pressure_pascal/">pressure_pascal</a></td>
<td>Pressure applied in Pascals</td>
</tr>
<tr>
<td><a href="proportion_ratio/">proportion_ratio</a></td>
<td>Ratio of model to biological proportions</td>
</tr>
<tr>
<td><a href="protein_binding/">protein_binding</a></td>
<td>Fraction bound to plasma proteins (0</td>
</tr>
<tr>
<td><a href="q_value/">q_value</a></td>
<td>False discovery rate corrected p-value</td>
</tr>
<tr>
<td><a href="quality_control_metrics/">quality_control_metrics</a></td>
<td>List of quality control measures and their values</td>
</tr>
<tr>
<td><a href="r_squared/">r_squared</a></td>
<td>R-squared value for regression models</td>
</tr>
<tr>
<td><a href="ratio/">ratio</a></td>
<td>Proportion or ratio of this cell type (0</td>
</tr>
<tr>
<td><a href="ratio_type/">ratio_type</a></td>
<td>Type of ratio specification (percentage, absolute, fold)</td>
</tr>
<tr>
<td><a href="real_time_data_sources/">real_time_data_sources</a></td>
<td>Sources of real-time data for model updating</td>
</tr>
<tr>
<td><a href="reference_value/">reference_value</a></td>
<td>Reference or control value for comparison</td>
</tr>
<tr>
<td><a href="references/">references</a></td>
<td>Literature references that describe, validate, or support this model</td>
</tr>
<tr>
<td><a href="replicate_count/">replicate_count</a></td>
<td>Number of experimental replicates used in assessment</td>
</tr>
<tr>
<td><a href="represented_cell_types/">represented_cell_types</a></td>
<td>List of cell types present in both model and biological system</td>
</tr>
<tr>
<td><a href="reproducibility/">reproducibility</a></td>
<td></td>
</tr>
<tr>
<td><a href="reproducibility_score/">reproducibility_score</a></td>
<td>Quantitative score (0</td>
</tr>
<tr>
<td><a href="rmse/">rmse</a></td>
<td>Root mean square error</td>
</tr>
<tr>
<td><a href="sensitivity/">sensitivity</a></td>
<td>Sensitivity/recall of the model (0</td>
</tr>
<tr>
<td><a href="sensor_integration/">sensor_integration</a></td>
<td>Sensors integrated for real-time monitoring</td>
</tr>
<tr>
<td><a href="sensors_integrated/">sensors_integrated</a></td>
<td>Sensors integrated into the device for monitoring</td>
</tr>
<tr>
<td><a href="shared_phenotypes/">shared_phenotypes</a></td>
<td>List of phenotypes present in both model and biological system</td>
</tr>
<tr>
<td><a href="shear_stress/">shear_stress</a></td>
<td>Shear stress applied in dyn/cm²</td>
</tr>
<tr>
<td><a href="similarity_score/">similarity_score</a></td>
<td>Quantitative similarity score (0</td>
</tr>
<tr>
<td><a href="single_cell_method/">single_cell_method</a></td>
<td>Method used for single-cell analysis (e</td>
</tr>
<tr>
<td><a href="size_range/">size_range</a></td>
<td>Typical size range of 3D structures</td>
</tr>
<tr>
<td><a href="software_platform/">software_platform</a></td>
<td>Software platform or programming language used</td>
</tr>
<tr>
<td><a href="spatial_context/">spatial_context</a></td>
<td>Description of spatial organization and context captured by the model</td>
</tr>
<tr>
<td><a href="special_features/">special_features</a></td>
<td>Additional special features of the device (e</td>
</tr>
<tr>
<td><a href="species/">species</a></td>
<td>The species of the animal used in the model system</td>
</tr>
<tr>
<td><a href="species_modeled/">species_modeled</a></td>
<td>Species for which the model is designed</td>
</tr>
<tr>
<td><a href="specificity/">specificity</a></td>
<td>Specificity of the model (0</td>
</tr>
<tr>
<td><a href="statistical_significance/">statistical_significance</a></td>
<td>Statistical measures of significance for the molecular similarity</td>
</tr>
<tr>
<td><a href="statistical_test/">statistical_test</a></td>
<td>Name of statistical test used</td>
</tr>
<tr>
<td><a href="stimulation_type/">stimulation_type</a></td>
<td>Type of mechanical stimulation applied</td>
</tr>
<tr>
<td><a href="strain/">strain</a></td>
<td>The specific strain of the animal used in the model system</td>
</tr>
<tr>
<td><a href="structured_concordance/">structured_concordance</a></td>
<td>Detailed structured assessment of concordance between the model system and th...</td>
</tr>
<tr>
<td><a href="studies/">studies</a></td>
<td></td>
</tr>
<tr>
<td><a href="substrate_type/">substrate_type</a></td>
<td>Type of culture substrate (e</td>
</tr>
<tr>
<td><a href="surface_treatment/">surface_treatment</a></td>
<td>Surface treatments or coatings applied to the device</td>
</tr>
<tr>
<td><a href="three_d_architecture/">three_d_architecture</a></td>
<td>Type of 3D architecture (spheroid, organoid, scaffold-based, etc</td>
</tr>
<tr>
<td><a href="threshold/">threshold</a></td>
<td>Acceptable threshold for this metric</td>
</tr>
<tr>
<td><a href="tissue_architecture/">tissue_architecture</a></td>
<td>Description of tissue-level architecture and organization</td>
</tr>
<tr>
<td><a href="tissue_modeled/">tissue_modeled</a></td>
<td>The specific tissue being modeled</td>
</tr>
<tr>
<td><a href="title/">title</a></td>
<td>Title of the referenced publication or dataset</td>
</tr>
<tr>
<td><a href="training_data_size/">training_data_size</a></td>
<td>Size of training dataset</td>
</tr>
<tr>
<td><a href="training_dataset_size/">training_dataset_size</a></td>
<td>Number of compounds in training dataset</td>
</tr>
<tr>
<td><a href="twin_scope/">twin_scope</a></td>
<td>Scope of digital twin (organ, patient, population)</td>
</tr>
<tr>
<td><a href="type/">type</a></td>
<td></td>
</tr>
<tr>
<td><a href="units/">units</a></td>
<td>Units of measurement for the assay result</td>
</tr>
<tr>
<td><a href="update_frequency/">update_frequency</a></td>
<td>Frequency of model updates based on new data</td>
</tr>
<tr>
<td><a href="url/">url</a></td>
<td>URL to access the publication</td>
</tr>
<tr>
<td><a href="validation_datasets/">validation_datasets</a></td>
<td>Datasets used for model training and validation</td>
</tr>
<tr>
<td><a href="volume/">volume</a></td>
<td>Volume of the compartment (L)</td>
</tr>
<tr>
<td><a href="width/">width</a></td>
<td>Width of the channel in micrometers</td>
</tr>
<tr>
<td><a href="year/">year</a></td>
<td>Publication year</td>
</tr>
</tbody>
</table>
<h2 id="enumerations">Enumerations</h2>
<table>
<thead>
<tr>
<th>Enumeration</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="AgeEnum/">AgeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="BiologicalOrganizationLevelEnum/">BiologicalOrganizationLevelEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="CaseOrControlEnum/">CaseOrControlEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="CellTypeEnum/">CellTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ChannelConfigurationEnum/">ChannelConfigurationEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="CocultureConfigurationEnum/">CocultureConfigurationEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ComplexityLevelEnum/">ComplexityLevelEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="CrossValidationMethodEnum/">CrossValidationMethodEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="DeviceMaterialEnum/">DeviceMaterialEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="DigitalTwinScopeEnum/">DigitalTwinScopeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="FeatureTypeEnum/">FeatureTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="FlowControlMethodEnum/">FlowControlMethodEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="IntegratedSensorEnum/">IntegratedSensorEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="InterfaceTypeEnum/">InterfaceTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="InterpretabilityLevelEnum/">InterpretabilityLevelEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="InvestigativeProtocolEnum/">InvestigativeProtocolEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="MechanicalStimulationTypeEnum/">MechanicalStimulationTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="MembraneTypeEnum/">MembraneTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="MicrofluidicArchitectureEnum/">MicrofluidicArchitectureEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="MLAlgorithmEnum/">MLAlgorithmEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="OrganEnum/">OrganEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="PBPKCompartmentEnum/">PBPKCompartmentEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="PredictionOutcomeEnum/">PredictionOutcomeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="PresenceEnum/">PresenceEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="RatioTypeEnum/">RatioTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="RelativeTimeEnum/">RelativeTimeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="SampleProcessingEnum/">SampleProcessingEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="SpeciesEnum/">SpeciesEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="StrainEnum/">StrainEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="StudyDesignEnum/">StudyDesignEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="SurfaceCoatingEnum/">SurfaceCoatingEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ThreeDArchitectureEnum/">ThreeDArchitectureEnum</a></td>
<td></td>
</tr>
</tbody>
</table>
<h2 id="types">Types</h2>
<table>
<thead>
<tr>
<th>Type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Boolean/">Boolean</a></td>
<td>A binary (true or false) value</td>
</tr>
<tr>
<td><a href="Curie/">Curie</a></td>
<td>a compact URI</td>
</tr>
<tr>
<td><a href="Date/">Date</a></td>
<td>a date (year, month and day) in an idealized calendar</td>
</tr>
<tr>
<td><a href="DateOrDatetime/">DateOrDatetime</a></td>
<td>Either a date or a datetime</td>
</tr>
<tr>
<td><a href="Datetime/">Datetime</a></td>
<td>The combination of a date and time</td>
</tr>
<tr>
<td><a href="Decimal/">Decimal</a></td>
<td>A real number with arbitrary precision that conforms to the xsd:decimal speci...</td>
</tr>
<tr>
<td><a href="Double/">Double</a></td>
<td>A real number that conforms to the xsd:double specification</td>
</tr>
<tr>
<td><a href="Float/">Float</a></td>
<td>A real number that conforms to the xsd:float specification</td>
</tr>
<tr>
<td><a href="Integer/">Integer</a></td>
<td>An integer</td>
</tr>
<tr>
<td><a href="Jsonpath/">Jsonpath</a></td>
<td>A string encoding a JSON Path</td>
</tr>
<tr>
<td><a href="Jsonpointer/">Jsonpointer</a></td>
<td>A string encoding a JSON Pointer</td>
</tr>
<tr>
<td><a href="Ncname/">Ncname</a></td>
<td>Prefix part of CURIE</td>
</tr>
<tr>
<td><a href="Nodeidentifier/">Nodeidentifier</a></td>
<td>A URI, CURIE or BNODE that represents a node in a model</td>
</tr>
<tr>
<td><a href="Objectidentifier/">Objectidentifier</a></td>
<td>A URI or CURIE that represents an object in the model</td>
</tr>
<tr>
<td><a href="Sparqlpath/">Sparqlpath</a></td>
<td>A string encoding a SPARQL Property Path</td>
</tr>
<tr>
<td><a href="String/">String</a></td>
<td>A character string</td>
</tr>
<tr>
<td><a href="Time/">Time</a></td>
<td>A time object represents a (local) time of day, independent of any particular...</td>
</tr>
<tr>
<td><a href="Uri/">Uri</a></td>
<td>a complete URI</td>
</tr>
<tr>
<td><a href="Uriorcurie/">Uriorcurie</a></td>
<td>a URI or a CURIE</td>
</tr>
</tbody>
</table>
<h2 id="subsets">Subsets</h2>
<table>
<thead>
<tr>
<th>Subset</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
<td></td>
</tr>
</tbody>
</table>












                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link"], "search": "../assets/javascripts/workers/search.d50fe291.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../assets/javascripts/bundle.50899def.min.js"></script>
      
    
  </body>
</html>