
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://monarch-initiative.github.io/namo/background/working-group-report/">
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.16">
    
    
      
        <title>Working group report - namo</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e37652d.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../css/ansi-colours.css">
    
      <link rel="stylesheet" href="../../css/jupyter-cells.css">
    
      <link rel="stylesheet" href="../../css/pandas-dataframe.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    <body dir="ltr">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="namo" class="md-header__button md-logo" aria-label="namo" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            namo
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Working group report
            
          </span>
        </div>
      </div>
    </div>
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/monarch-initiative/namo" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path fill="currentColor" d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="namo" class="md-nav__button md-logo" aria-label="namo" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    namo
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/monarch-initiative/namo" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path fill="currentColor" d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../use-cases/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Use Cases
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    How-to
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            How-to
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../how-to/curate/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../how-to/curate-organoid-paper/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Curate Organoid Papers
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../how-to/curate-organ-on-chip-paper/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Curate Organ-on-Chip Papers
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../elements/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Schema
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../about/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    About
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


  <h1>Working group report</h1>

<p>CATALYZING THE
DEVELOPMENT AND
USE OF NOVEL
ALTERNATIVE
METHODS</p>
<p>December 2023
Report to the Advisory
Committee to the Director</p>
<p>Catalyzing the Development and Use of Novel Alternative Methods</p>
<p>Preamble</p>
<p>The 21st century is a time of expeditious technological acceleration. Increasing
use of new and improved biomedical technologies, such as gene editing,
artificial intelligence (AI), and induced pluripotent stem cells, are
fundamentally changing the way science is done. When successfully combined
with traditional research methodologies, these novel alternative methods
(NAMs) enable research to be done more quickly, by more researchers, at a
more affordable cost. Importantly, these advances can also drive innovation in
scientific methodologies themselves, opening doors to new scientific approaches,
including complementary and more effective non-animal-based approaches.
Taken together, harnessing the power of emerging technologies to advance
novel scientific approaches holds tremendous promise for helping us better
understand fundamental biology to advance human health.</p>
<p>Successful deployment of NAMs, whether for conducting basic research,
uncovering disease mechanisms, or translating knowledge into products or
practice, relies on bringing together many disciplines, technologies, data, and
areas of expertise. Interdisciplinary collaboration drives creativity and
innovation, and teams of people coming from different backgrounds can create
out-of-the-box solutions that would otherwise never be imagined. Each sector
within the biomedical research enterprise has a role to play in catalyzing the
development and use of NAMs. By integrating these perspectives and needs
early in technology conception, researchers can develop NAMs that provide
high-quality, reproducible findings with the highest relevance to human
biology.</p>
<p>ACKNOWLEDGEMENTS
Our Working Group recognizes the many contributions of the public, patients, research participants,
researchers, and broader scientific enterprise in bringing this endeavor to fruition. We are thankful for
the time and expertise contributed to our many discussions and information collection, as well as
holding us accountable for ensuring our findings can spur advances across the areas of opportunities
that lie ahead.</p>
<p>We also would also like to thank all of those that have been at the forefront of developing and using
NAMs for the past several decades. We have learned tremendously from these efforts as this work has
been truly catalytic in terms of spurring both tremendous scientific opportunity and inspiration for
generations to come regarding the promise of innovation.</p>
<p>Finally, we would like to thank the many individuals who have supported the analyses, writings, and
logistics of this group. We are extremely grateful to our outstanding Executive Secretaries, Dr. Brittany
Chao and Dr. Jessica Creery, who have worked tirelessly to lead teams in support of these functions.
We also would like to acknowledge the contributions of:</p>
<p>•  Abigail McElroy (Emory University) for her assistance in conceptualizing mechanisms for</p>
<p>collecting public input and analyzing the resulting data;</p>
<p>•  Gabriela Zamora (NIH), Maxim Litvinov (Stanford), and Daniel Braslavsky (Stanford) for co-</p>
<p>chair logistical support; and</p>
<p>•  The NIH Office of Science Policy team who provided support for meetings, communication and</p>
<p>outreach, and other Working Group activities as needed.</p>
<p>4</p>
<p>WORKING GROUP MEMBERS</p>
<p>Howard Y. Chang, MD, PhD (co-chair)
Stanford University</p>
<p>Linda Griffith, PhD
Massachusetts Institute of Technology</p>
<p>Lyric Jorgenson, PhD (co-chair)
National Institutes of Health</p>
<p>Ranu Jung, PhD
University of Arkansas</p>
<p>Antonio Baines, PhD
NC Central University/University of North
Carolina</p>
<p>Szczepan W. Baran, VMD, MS
VeriSIM Life</p>
<p>Wendy Chapman, PhD
University of Melbourne</p>
<p>Myrtle Davis, DVM, PhD, ATS
Bristol Myers Squibb</p>
<p>EX OFFICIOS MEMBERS
Danilo A. Tagle, PhD
National Institutes of Health</p>
<p>Namandjé Bumpus, PhD
Food and Drug Administration</p>
<p>EXECUTIVE SECRETARIES</p>
<p>Brittany Chao, DPhil
National Institutes of Health</p>
<p>Arnold Kriegstein, MD, PhD
University of California, San Francisco</p>
<p>Nancy E. Lane, MD
University of California, Davis</p>
<p>Kelly Metcalf Pate, DVM, PhD
Massachusetts Institute of Technology</p>
<p>Sergiu Pasca, MD
Stanford University</p>
<p>Gordana Vunjak-Novakovic, PhD
Columbia University</p>
<p>Maureen Gwinn, PhD
Environmental Protection Agency</p>
<p>Jessica Creery, PhD
National Institutes of Health</p>
<p>5</p>
<p>TABLE OF CONTENTS</p>
<p>EXECUTIVE SUMMARY</p>
<p>....................................................................................................................................
 7</p>
<p>SECTION I.  CHARTING THE COURSE FOR THE DEVELOPMENT AND USE OF NAMs</p>
<p>........................................</p>
<p>10</p>
<p>SECTION II.  CAPITALIZING ON THE OPPORTUNITY OF NAMs
SECTION III. HIGH PRIORITY NEEDS &amp; RECOMMENDATIONS</p>
<p>........................................................................</p>
<p>.........................................................................</p>
<p>13
 16</p>
<ol>
<li>Combinatorial NAMs</li>
</ol>
<p>........................................................................................................................... 16</p>
<ol>
<li>Interoperable, Reliable Datasets</li>
</ol>
<p>..........................................................................................................</p>
<p>18</p>
<ol>
<li>Effective Technology Dissemination And Interconnection</li>
</ol>
<p>.....................................................................</p>
<p>20</p>
<ol>
<li>Comprehensive Training</li>
</ol>
<p>.....................................................................................................................</p>
<p>21</p>
<ol>
<li>Multidisciplinary Teams</li>
</ol>
<p>......................................................................................................................</p>
<p>22</p>
<ol>
<li>Socially Responsible Technologies</li>
</ol>
<p>.......................................................................................................</p>
<p>23</p>
<ol>
<li>Coordinated Infrastructure</li>
</ol>
<p>..................................................................................................................</p>
<p>25</p>
<p>SECTION IV.  SUCCESSFUL INTEGRATION: TRANSFORMATIVE PROJECTS .......................................................</p>
<p>26</p>
<p>Transformative Project Exemplar 1: Leveraging In Vitro and In Silico Techniques for Actionable Insights in
.....................................................................................................................
Complex Biological Models</p>
<p>26</p>
<p>Transformative Project Exemplar 2: Uncovering New Understanding of Neuropsychiatric Disorders</p>
<p>..........</p>
<p>27</p>
<p>Transformative Project Exemplar 3: Combining In Vitro, In Vivo (Clinical And Preclinical), and In Silico
Approaches to Minimize Dependence on Preclinical In Vivo Models</p>
<p>.........................................................</p>
<p>28</p>
<p>Transformative Project Exemplar 4: Combinatorial Approaches to Improve Treatment of Chronic
Inflammatory Conditions</p>
<p>........................................................................................................................</p>
<p>29</p>
<p>Transformative Project Exemplar 5: Data Integration Across NAMs, Traditional Models, and the Clinic to
 ...........................................................................
Emulate Patient-Specific Tumor-Immune Environments</p>
<p>30</p>
<p>SECTION V.  CONCLUDING REMARKS</p>
<p>............................................................................................................</p>
<p>32</p>
<p>APPENDICES</p>
<p>..................................................................................................................................................</p>
<p>33</p>
<p>APPENDIX A – Definitions and Terms</p>
<p>.......................................................................................................</p>
<p>33</p>
<p>APPENDIX B – NIH Request for Information (RFI)</p>
<p>......................................................................................</p>
<p>36</p>
<p>APPENDIX C – Agenda and Participants, August 2023 Public Meeting</p>
<p>........................................................</p>
<p>40</p>
<p>6</p>
<p>EXECUTIVE SUMMARY
Scientific breakthroughs are often propelled by the introduction of new technologies that can transform how
scientists study health and disease. While some technologies are truly novel and disruptive, the majority are
discovered through a continuous and iterative cycle of development, standardization, validation, and uptake.
As technological capabilities have improved over the last decade, they equip researchers with a new set of
methods that offer unique precision and can potentially reduce reliance on animal model systems for specific
types of studies. These “Novel Alternative Methods” (NAMs, sometimes defined as New Approach
Methodologies, Non-Animal Methods, or New Alternative Methods) provide a complementary approach to
traditional models while offering tremendous promise for enhancing understanding of the human system and
for more effectively treating human conditions.</p>
<p>The burgeoning field of NAMs is quite diverse and
each has its own unique strengths and weaknesses
depending on the context of use. Accordingly, in
December 2022, Acting NIH Director Dr. Lawrence
Tabak convened the Working Group, comprising
experts across disciplines and sectors to assist NIH
in prioritizing the development and use of NAMs
with the highest potential for catalyzing biomedical
research. The Working Group were asked to
identify the landscape of NAMs currently being
deployed; assess their strengths and limitations;
and characterize the types of research in which
they hold the most promise of complementing
and/or replacing animal studies. Ultimately these
findings are intended to inform the Advisory
Committee to the Director (ACD) in providing
recommendations to the NIH for priority settings
and future investments.</p>
<p>NAMs DEFINED (see Appendix A for
additional definitions)</p>
<p>In silico: Experiments performed by computing
platforms or custom hardware, encompassing
mathematical modeling and simulation,
machine learning, and other computational
techniques.</p>
<p>In chemico: Experiments performed on
biological molecules, such as proteins and DNA,
outside of cells, which may be used to study
how these molecules interact with each other
and with drugs.</p>
<p>In vitro: Experiments performed on cells
outside of the body, including various types of
cell, organoid, and tissue culture techniques.</p>
<p>The Working Group worked intensively throughout 2023, meeting with developers and users of NAMs to
explore not only the current context of use, but also opportunities for future development and deployment.
For the purposes of the WG’s discussions and findings, NAMs were classified into three general categories of
technologies: computational modeling and predictive technologies (in silico), cell-free methods and assays (in
chemico), and cell-tissue-organoid culture models (in vitro). Each model offers unique strengths that, when
utilized individually or in combinations, expands researchers’ toolboxes to improve upon and answer
previously unanswerable biomedical research questions, and to ask new questions. For NAMs to be effective
and reliable, these strengths and weaknesses should be assessed prior to incorporation into a study, and
context of use should be considered in NAMs development.</p>
<p>It was clear throughout the WG’s deliberations that NAMs are already incredibly valuable for conducting basic
research, uncovering pathophysiological mechanisms, and translating discovery and knowledge into products
or practice. Numerous areas of biomedical research are already benefitting from NAMs, including cancer,
diabetes, cardiovascular disease, Alzheimer’s disease, mental illness, infectious disease, rare diseases, and
more. However, no one individual NAM can currently fully recapitulate human physiology. For NAMs to reach
their full potential, much more needs to be done to unite and interconnect the underlying disciplines,
technologies, data, and areas of expertise. By integrating these perspectives and needs early in technology</p>
<p>7</p>
<p>conception, researchers can develop NAMs that provide high-quality, reproducible findings with the highest
relevance to human pathophysiology.</p>
<p>Recognizing that integration is the key for delivering on the transformative promise of NAMs, the Working
Group identified seven thematic clusters of high priority needs that should be addressed in moving forward.
Ideally, these needs would be addressed in tandem and are depicted in the figure below.</p>
<p>8</p>
<p>In support of these needs, the Working Group identified key recommendations and activities that NIH should
pursue in each of these areas:</p>
<p>RECOMMENDATIONS TO CATALYZE THE DEVELOPMENT AND USE OF NAMs</p>
<p>Recommendation 1.</p>
<p>Prioritize the development and use of combinatorial NAMs.</p>
<p>Recommendation 2.</p>
<p>Establish resources, infrastructure, and collaborations to promote the use of
interoperable, reliable, and well curated/high quality datasets produced
from research using NAMs.</p>
<p>Recommendation 3.</p>
<p>Promote effective dissemination and interconnection of NAMs technologies.</p>
<p>Recommendation 4.</p>
<p>Recommendation 5.</p>
<p>Recommendation 6.</p>
<p>Recommendation 7.</p>
<p>Invest in comprehensive training to bolster continuous advances in NAMs
development and use.</p>
<p>Facilitate multidisciplinary teams with expertise across technologies and the
lifecycle of NAMs development and use.</p>
<p>Promote social responsibility in both the creation and deployment of NAMs
across the research lifecycle.</p>
<p>Support and maintain coordinated infrastructure to catalyze effective and
responsible NAM development and use.</p>
<p>The recommendations above are not mutually exclusive and hinge upon the importance of putting together
diverse, multi-disciplinary teams with the right complementary knowledge. Breaking down silos includes
setting up collaborations between groups (e.g. disciplines, sectors), training scientists in a multi-disciplinary
fashion, creating standardized language to communicate across specialties and sectors, and building and
maintaining an infrastructure to foster data interoperability and integrated models. As all new scientific
advances build upon prior scientific advances, it is essential to keep researchers who work with traditional
models engaged throughout this endeavor. Similarly, as the ultimate goal is the advancement of medicine, it is
important to consult with the clinicians who will put this new knowledge into use, to make sure that the
results produced are translationally relevant, heed regulatory guidelines, and can be integrated into existing
workflows in clinical settings.</p>
<p>While this report is focused on NIH, the vast and expansive applicability of the recommendations necessitates
participation across sectors. Each sector has a role to play in supporting an integrated ecosystem for catalyzing
the development and use of NAMs. In addition to NIH, the Working Group expect this report to serve as a
basis for other stakeholders to take in consideration in the development and myriad uses of NAMs. The large-
scale uptake of any new technology by researchers, practitioners, and patients will require a cultural shift,
while deploying new technologies will require communication, training, and building confidence in NAMs
among the research and technology development community and the public.</p>
<p>9</p>
<p>SECTION I. CHARTING THE COURSE FOR THE DEVELOPMENT AND USE OF NAMs
From its foundation to the present day, NIH has funded research into the development and application of
novel technologies and approaches. While not traditionally categorized as “novel” or “alternative”, the NAMs
field itself has seen tremendous growth over the past 15 years alongside ever-expanding technological
capabilities. Often, NIH-funded researchers use these methods to help guide, or in tandem with animal
studies, bolster evidence for their conclusions in going from simpler to more complex models. In many cases,
NAMs allow scientists to control variables and establish clearer roles for the building blocks of biological
systems, while research in animal models is critical to understanding just how these fundamental pieces
interact in a living organism as it behaves over time in its environment. These NAMs are applied and continue
being developed in a wide range of areas of basic and clinical research, including cancer, diabetes,
cardiovascular disease, Alzheimer’s disease, mental illness, infectious disease, and rare and genetic diseases.</p>
<p>Historical Context. NIH-funded researchers have used NAMs extensively for scientific and medical discovery,
and they continue to hold great promise for the future. As NAMs become increasingly sophisticated, the
prospect of refining, reducing, and replacing the use of animal models in research (also referred to as the
3Rs)1  becomes more feasible. Accordingly, NIH continues to invest into identifying and developing
appropriate biological systems, including NAMs, to maximize research translation. However, each model has
strengths and limitations, which may vary depending on the specific research questions being addressed. By
strategically increasing NIH’s portfolio by investment into NAMs, NIH can provide researchers with tools that
are complementary to, and potentially replacements of, traditional models that hold great promise in
establishing more accurate, relevant, and reliable research into human health and disease.</p>
<p>It is important to acknowledge the many long-standing efforts within the biomedical research community
along with a recent convergence of factors that highlight this as an ideal time to invest strategically in the
development and use of NAMs.  Recent reports have demonstrated the need for such strategic prioritization,
including the May 2023 release of a congressionally requested and NIH funded, National Academies of
Sciences, Engineering, and Medicine (NASEM) analysis on the state of the science for nonhuman primate (NHP)
model systems, including the current role of NHPs in biomedical research, future needs for NHPs, and
opportunities for new approach methodologies to complement or reduce NHPs in biomedical research.2 The
analysis concluded that while there are currently no alternative approaches that can replace NHP models to
answer research questions that require complete multiorgan interactions and integrated biology, there is
value in working towards that goal. The NASEM analysis noted that efforts to reduce reliance on NHPs in
biomedical research will require direct interaction and collaborative research among investigators using NHP
models and those developing in vitro and in silico approaches to expand the applicability of NAMs to research
questions for which NHPs are currently needed. The current Working Group report and recommendations also
build on recommendations by the NIH Advisory Committee to the Director (ACD) Working Group on Enhancing
Rigor, Transparency, and Translatability in Animal Research, which were adopted by the ACD, to explore
alternative approaches and to improve selection, design, and relevance of animal models.3  NIH recognizes
that to support rigorous and impactful science, the choice of model or integrated models and methodologies
used in biomedical research studies must be selected based on scientific appropriateness and relevance to
human health and disease.</p>
<p>1 caat.jhsph.edu/russell-and-burchs-principles-of-humane-experimental-techniques/
2 nationalacademies.org/our-work/nonhuman-primate-model-systems-state-of-the-science-and-future-needs
3 acd.od.nih.gov/documents/presentations/06112021_ACD_WorkingGroup_FinalReport.pdf</p>
<p>10</p>
<p>Working Group Approach. Acting NIH Director Dr. Lawrence Tabak proposed the initial NAMs Working Group
charge to the ACD in November 20224 and formally launched the Working Group in December 2022.5 Led by
Drs. Howard Chang and Lyric Jorgenson, the Working Group met monthly from January to November 2023,
reviewing the state-of-the-science, assessing current and planned initiatives, and meeting with experts across
disciplines and sectors.</p>
<p>Early on, the WG’s deliberations clearly showed that
the breadth and diversity of NAMs warranted
additional mechanisms for feedback, especially to
determine which areas of research are currently
underserved or may benefit from more sophisticated
or easier to use NAM technologies. Accordingly, NIH
held a public workshop in August 2022 to discuss
current uses, challenges, and opportunities of NAMs,
and included representatives from academic, non-
profit, private, and government partners.6 The
workshop participants brought expertise across
scientific disciplines and experience in both
developing and using NAMs. To capture input from
the public more broadly, including non-traditional
partners, the NIH issued a public Request for
Information (RFI) (NOT-OD-23-140) to hear more
from these stakeholders.7 NIH shared the 85
responses with the Working Group and the public,
which they reviewed and used to inform their
deliberations and recommendations (See APPENDIX
B).</p>
<p>CHARGE TO THE ACD NAMs WORKING GROUP</p>
<p>•</p>
<p>Identify the types of alternative methods being
developed for use in biomedical research and
assess their general strengths and weaknesses
for studying human biology, circuits, systems,
and disease states;</p>
<p>•  Characterize the types of research, condition,
or disease for which alternative methods are
most applicable or beneficial; and</p>
<p>•  Articulate high-priority areas for NIH</p>
<p>investment in the use and development of
novel alternative methods with human
applicability to:</p>
<p>o  Advance progress into understanding specific</p>
<p>biological processes or states; and</p>
<p>o  Augment the tools and capabilities for</p>
<p>biomedical research to complement and/or
potentially replace traditional models.</p>
<p>Scoping the Task. It became increasingly clear that challenges facing progress in developing and using
NAMs begin with a fragmented system for how to conceptualize, define, and standardize these technologies.
Clear and interoperable definitions are needed to catalyze the field but also promote use of NAMs more
broadly across the biomedical research ecosystem. In the WG’s discussions, they focused on the following
categories of NAMs, defined as follows: in silico methods, experiments performed by computing platforms or
custom hardware, encompassing mathematical modeling and simulation, machine learning, and other
computational techniques; in chemico methods, experiments performed on biological molecules, such as
proteins and DNA, outside of cells, which may be used to study how these molecules interact with each other
and with drugs; and in vitro methods, experiments performed on cells outside of the body, including various
types of cell, organoid and tissue culture techniques.</p>
<p>While the Working Group acknowledged that there is a broad and diverse set of tools and techniques that
could fall under the rubric of “alternatives”, they agreed about an initial focus on areas in which emerging
technological advances can provide new or improved models for interrogating biological processes or states.
For example, NIH-funded research in single cell eukaryotes (e.g., yeast) and invertebrate animals (e.g., fruit
flies, C. elegans) has been transformative and have reduced reliance on traditional vertebrate animal models.</p>
<p>4 acd.od.nih.gov/documents/presentations/11032022_Biomedical_Research.pdf
5 acd.od.nih.gov/documents/presentations/12082022_Proposed_ACD_Novel_Alternative.pdf
6 osp.od.nih.gov/events/nih-workshop-on-catalyzing-the-development-of-novel-alternatives-methods/
7 grants.nih.gov/grants/guide/notice-files/NOT-OD-23-140.html</p>
<p>11</p>
<p>Studies in these models have resulted in nine Nobel prizes in Physiology and Medicine because some of the
fundamental biological mechanisms are conserved from these animals all the way to humans, revealing
detailed insights into cancer,8 the mechanisms of sleep,9 memory and learning,10 and the brain functions,11
amongst others. Additionally, research studies can be increasingly conducted with human participants and NIH
remains committed to direct epidemiological and non-invasive studies involving humans. While these
approaches play an important role in reducing the number of animals necessary for research, the focus of this
assessment is on a subset of NAMs that can take advantage of new technologies, more faithfully represent
human pathophysiology and serve as models of tissue/organ injury, regeneration, disease and treatment.</p>
<p>Finally, the WG scoped its task on NAMs to focus on those with the greatest potential for human relevance.
The NIH mission is to seek fundamental knowledge about the nature and behavior of living systems and the
application of that knowledge to enhance health, lengthen life, and reduce illness and disability. Thus,
clinically relevant questions should serve as the driver for the science, and the development of NAMs and their
use, whether alone or in concert with other NAMs and more traditional models, occur in response to those
questions.12  With in vitro methods, the use of human cell lines and human tissue surrogates can increase
human relevance and simultaneously provide potential to reduce the use of traditional animal models.
Further, in silico methods can guide in vitro and in vivo studies and ones that use data from patients, or model
and predict conditions and treatments can increase clinical relevance and translatability of findings. With this
said, and given the complexity of human biology and disease, it is challenging to envision a near-term future
where animal studies are not necessary for improving public health and protecting the public and patients
from unintended harm. The value of any modeling approach is based on the assertion that known similarities
between the model and the subject matter permit conclusions that additional features observed in the model
will also be observed in the domain to which the model is applied.13 Some NAMs, such as computational
models, may reveal unexpected and complex causal relationships among variables that are unlikely to emerge
exclusively in animal studies.14  In this context, animal models may be applied in a complementary fashion to
confirm novel hypotheses. The continued development of NAMs will improve the ability to progressively
decrease that dependence even more and improve the ability to translate fundamental biomedical research to
patients.</p>
<p>8 nobelprize.org/prizes/medicine/1995/summary/
9 nobelprize.org/prizes/medicine/2017/press-release/
10 nobelprize.org/prizes/medicine/2000/kandel/facts/
11 nobelprize.org/prizes/medicine/2002/summary/
12 nationalacademies.org/our-work/variability-and-relevance-of-current-laboratory-mammalian-toxicity-tests-and-expectations-for-new-approach-methods--nams--
for-use-in-human-health-risk-assessment
13 Bartha PFA (2010) By parallel reasoning: the construction and evaluation of analogical arguments.
Oxford University Press, New York
14 M. Ferrante et al. Molecular Psychiatry (2019) 24:479–483</p>
<p>12</p>
<p>SECTION II. CAPITALIZING ON THE OPPORTUNITY OF NAMs
NIH funds research using animal and non-animal approaches to investigate biology from molecules, to
systems, to whole organisms and how they interact with their environments. Depending on the biological
system or disease state, different combinations of methods may be required to provide the strongest body of
evidence, so when and how to replace animal models with NAMs needs to be determined by the
appropriateness of the model for the research hypothesis. While NAMs have grown increasingly sophisticated
in mimicking human biology, it is important that the community fully understand their strengths and
limitations, as well as areas of opportunity given the current state of science.</p>
<p>The Working Group would be remiss if they did not acknowledge that significant efforts have been dedicated
to improving and reducing use of animals by enhancing their translational relevance, supporting their greater
reproducibility, and relieving pain and discomfort. NIH is committed to replacing, reducing, and refining the
use of animals in biomedical research. Significant effort and investment have been dedicated to developing
capabilities that advance the ability to model human biology without the use of animals. Currently, these
capabilities are focused on investigating basic biological and disease mechanisms, while some are beginning to
support toxicology testing. The rapidly evolving understanding of disease biology complemented by the
development of increasingly complex non-animal models will converge to support a future that is more
effective at supporting research that improves public health while equipping researchers with capabilities to
choose a model most suitable for study.</p>
<p>The promise of NAMs is recognized by both researchers and the public, with numerous federal and private
sector initiatives underway both domestically and abroad. There are many existing reviews detailing the
strengths and weaknesses of various NAMs that the Working Group will not recapitulate as part of this report.
However, to augment the report findings, the Working Group  reference a recent NASEM non-human primates
and NAMs consensus report15 to which several of the Working Group members contributed, which describes
examples of NAMs, recent changes in the regulatory landscape that are shaping the use of alternatives for
studies involving human safety, and needs and opportunities for facilitating collaboration between non-human
primate researchers and those who develop and use NAMs. Importantly, in 2022, the U.S. Congress directed
the NIH to assess its current NAMs portfolio which was shared with the WG group to inform their
understanding of potential gaps and needs. These analyses demonstrate increasing use of NAMs in NIH
funded projects, and that NAMs are being developed and applied in a wide range of areas, including cancer,
diabetes, cardiovascular disease, Alzheimer’s disease, mental illness, infectious disease, rare diseases, and
more. Rather than focus on specific diseases or conditions, the Working Group decided that their
recommendations would be most informative if they could be applied to the entirety of the NIH portfolio.
Thus, the WG’s findings and recommendations are disease agnostic and focus more broadly across various
stages of the research lifecycle, as described in further detail below.</p>
<p>Conducting Basic Research. NAMs have shown immense promise for basic research studies in which
controlling variables and delineating precise building blocks of biological systems are needed. NAMs such as in
vitro human cell cultures or single cell organisms are often utilized by researchers to elucidate fundamental
functions of cells and the basic biological mechanisms of cell functions. Complex in vitro models like 3-D tissue
cultures can also provide information on how cells interact with each other in a localized environment, such as
how different cell types in the same tissue interact or how cells like immune cells can move through 3-D
structures. In silico models can identify novel molecules, simulate target engagement and drug-drug
interactions (DDI), predict combinatorial therapy efficacy and toxicity, and guide preclinical studies and patient</p>
<p>15 nationalacademies.org/our-work/nonhuman-primate-model-systems-state-of-the-science-and-future-needs</p>
<p>13</p>
<p>stratification strategies. In silico models and in vitro NAMs typically recreate isolated components of the body
(i.e., individual cells, tissues, or organs) and multi-tissue units recapitulating important aspects of specific
component interactions (such as organ-organ communication, inflammation, infection, or cancer). As such,
while researchers have made tremendous strides in more sophisticated, multi-physiological system NAM
models, they currently cannot substitute for the whole organism. The Working Group emphasizes that this is
not a unique challenge for NAMs, as most animal models also fail to replicate the complexity of the human
body as an integrated system. That said, this is an area of intense opportunity moving forward.</p>
<p>Uncovering Human Patho/Physiological Mechanisms. As alluded to in the previous section, an area of
immense opportunity is use of suitable NAMs for studying the complexities of human biology and disease,
especially for multi-organ interactions; the intersection of brain, cognition, and behavior; the body’s response
to infectious disease; intricacies of human development across the lifespan; and characterizing long-term,
systemic, and developmental health effects of environmental and drug exposures. Traditional animal models
remain the gold standard for studying many aspects of human health and disease but have similar limitations
in terms of replicating the complexity of human physiology (and even more so for predicting the course of
human disease). Integrated NAMs could provide researchers with a host of new approaches to study human
physiology, which could be used in tandem with human disease datasets as an integrated and predictive
model. NAMs are also uniquely suited to make accessible entirely new areas of research that are difficult to
study in living people and animals (such as pediatric conditions and rare and complex diseases). Development
of more physiologically relevant NAMs could be deployed for better prediction of disease outcomes for these
conditions as well as assist in the development of therapeutic agents. For example, current animal models
often lack sufficient predictive value which has slowed drug development for the treatment of nervous system
disorders,16 and utilization of suitable NAMs derived from human data and cells could enhance and expedite
predictive studies. In areas that are unable to move forward due to lack of appropriate model accessibility,
such as with NHPs in the face of the current NHP supply shortage, the biomedical research community is
looking for ways to use NAMs to supplement and reduce reliance on these extremely limited and invaluable
resources.</p>
<p>Translating Knowledge into Products or Practice. The use of NAMs currently is perhaps most evident in
translational disciplines, especially for drug development and discovery, as they can provide testing at scale.
Sometimes, researchers need to explore hundreds, thousands, or even millions of possible targets, such as
when searching for new drug compounds and exploring their efficacies and toxicities. The number of assays
that can be performed is generally inverse to the complexity of the system, meaning that it is much more
effective, rapid, and cost-effective to use NAMs such as in vitro cell and tissue models to run these screens
than to test each drug using animal models. NAMs may also be transformative for studying rare conditions in
which disease incidence is low or inadequate, no traditional models exist, or large clinical trials are not
feasible. However, like traditional models, NAMs must also be subject to robust qualification and testing
measures to ensure relevance in the context of their proposed use, and to ensure their predictive value in
regard to safety and efficacy in the case of therapeutics in humans. Improvements in these capabilities are
clear as there is evolving regulatory guidance regarding these approaches. When breadth or cost-effective
exploration is the goal, high throughput NAMs may be a more practical model.</p>
<p>Toxicology research and testing studies have been of increasing focus. This is commensurate with a renewed
investment in the development and adoption of NAMs for these types of studies. Toxicology testing aims to</p>
<p>16 nationalacademies.org/our-work/therapeutic-development-for-nervous-system-disorders-in-the-absence-of-animal-models-a-workshop</p>
<p>14</p>
<p>identify and characterize human health hazards associated with environmental/drug exposures and develop
approaches for mitigating or eliminating risk. NIH already has several investments in this domain, such as:</p>
<p>•  Toxicology in the 21st Century (Tox21)17, a U.S. federal collaboration whose partners include the</p>
<p>National Toxicology Program (NTP)18, National Institute of Environmental Health Sciences (NIEHS),
National Center for Advancing Translational Sciences (NCATS), U.S. Environmental Protection Agency
(EPA), and U.S. Food and Drug Administration (FDA). Tox21 enables new high-throughput and
alternative methods to evaluate chemicals efficiently.</p>
<p>•  The NTP Interagency Center for the Evaluation of Alternative Toxicological Methods (NICEATM),19 an
office within NIH/NTP, evaluates alternatives to animal use for chemical safety testing with a focus on
scientific publishing. Additionally, NICEATM runs the Interagency Coordinating Committee on the
Validation of Alternative Methods (ICCVAM).20</p>
<p>Developing NAMs for standardized testing methods is uniquely tractable due to the long history in
characterizing toxicological health effects, the consistency of the aims of those methods, the availability of
historical data from many years of testing from which to develop computational models, and the ability to
compare new approaches to traditional animal-based approaches.</p>
<p>Prioritizing Rigor and Reproducibility: Unique Considerations for NAMs. Rigorous and reproducible study
design, as well as promulgation of study findings and technology use, remain a challenge across the research
enterprise. It is important to acknowledge that any type of research study must prioritize integration of robust
research practices to maximize scientific integrity. The development of NAMs alone will not solve the problem
and animal models must also improve (as discussed in greater detail by the ACD Working Group on Enhancing
Rigor, Transparency, and Translatability in Animal Research).21 However, the various NAMs, like all
technologies and models, have their own unique challenges that, if left unaddressed, can limit their utility and
ability to advance understanding of human health. The lack of consensus in terms of definitions and standards,
as well as appropriate technological benchmarks, remain a challenge for emerging technologies and for
integration and deployment of various NAMs. These ambiguities should be considered upfront in NAMs
development with appropriate caveats, and described in technology use. For example, it is important to
account for biological variability from donor source, especially for in vitro methods that can provide patient-
specific studies and investigate effects of sex, ethnic background, age and state of health or disease. Likewise,
it is key to mitigate selection bias and perpetuation of biases., particularly from in silico methods). There are
also specific equity considerations that should be accounted for when using NAMs, such as understudied
diseases, heterogenous populations, and lifespan. When paying special attention to human applicability,
development and use of the models must take into consideration the complexity of interacting biological and
environmental factors. However, these challenges also represent immense opportunity: the ability to
incorporate human variability into study design offers researchers the potential to represent human diversity
in a way that may not be readily accessible otherwise.</p>
<p>17 ntp.niehs.nih.gov/whatwestudy/tox21/index.html
18 ntp.niehs.nih.gov/
19 ntp.niehs.nih.gov/whatwestudy/niceatm/index.html
20 ntp.niehs.nih.gov/whatwestudy/niceatm/iccvam/index.html
21 acd.od.nih.gov/working-groups/eprar.html</p>
<p>15</p>
<p>SECTION III. HIGH PRIORITY NEEDS &amp; RECOMMENDATIONS
It is clear that NAMs are already incredibly valuable for conducting basic research, uncovering
patho/physiological mechanisms, and translating knowledge into products or practice. However, no one
individual NAM can fully recapitulate or simulate human physiology, and the goal should be an optimized,
clinically relevant predictive model. Better comparisons between animal models, NAMs, and humans are
needed to understand the full potential of different NAM systems, identify the best combinations in which to
use the different systems and technologies to answer complex biological questions, and ultimately increase
translatability. Research support is needed to compare relevant animal, NAMs and human models to validate
translational potential, reduce reliance on singular methods, and identify strengths and weaknesses in hybrid-
model approaches.</p>
<p>For NAMs to reach their full potential, more must be done to unite and interconnect disciplines,
technologies, data, and areas of expertise. By integrating these perspectives and needs early in technology
conception, researchers can develop NAMs that provide high-quality, reproducible findings with the highest
relevance to human biology. Recognizing that integration is the key for delivering on the transformative
promise of NAMs, the Working Group identified seven thematic clusters of high priority needs that should be
addressed moving forward (see figure). While the scope of this report was focused on NIH, the Working Group
emphasizes that each sector has a role to play in supporting an integrated ecosystem and should address
these needs in tandem.</p>
<p>HIGH PRIORITY NEED 1: COMBINATORIAL NAMs</p>
<p>Opportunity/Need. More sophisticated NAMs will be achieved by catalyzing approaches that integrate and
combine different methods. Technology combinatorial effect is a phenomenon where the integration or
combination of different technologies or their components results in a more significant impact than the sum
of their individual effects. In the context of NAMs, the strategic combination of NAMs can lead to
breakthroughs that would not have been possible with any single NAM in isolation. This type of interaction
and integration of technologies can lead to new opportunities, increased efficiency, improved performance, or
the development of entirely new tools.</p>
<p>Hybrid or integrated NAMs can be used for closed-loop systems (i.e., either data-driven or model driven)
throughout a research cycle and address the “black box” of in silico methods. For example, data from patients
can be used to generate hypotheses by training large scale models, which can be validated through in vivo and
in vitro models, which can then inform and generate new hypotheses for in silico models and for human
health. Combining modeling and machine learning, a hybrid AI approach, increases the ability to explain data
resulting from these NAMs. Successful combinatorial approaches may include any of the following:</p>
<p>•  Only newly developed NAMs</p>
<p>•  A mix of previously developed NAMs</p>
<p>•  A mix of previously developed NAMs and newly developed NAMs</p>
<p>•  NAMs and animal model systems</p>
<p>The desired outputs of the technologies and their successful outcomes need to be determined ahead of model
development. There is a need to bring stakeholders together to identify what makes the most sense to fund
by identifying the research questions based on what is not yet well-served by current methodologies,
including animal research. It is important to include experts in different technologies so the decision on what</p>
<p>16</p>
<p>to invest in is rooted in what is possible. When NIH reviews proposals to evaluate what to further develop,
reviewers need to be cognizant of and if needed trained to appreciate the goal and technology behind the
NAMs along with their usability for addressing specific research questions.</p>
<p>For successful integration across approaches, there is a need for multidisciplinary teams (see High Priority
Need 5) with access to interoperable high-quality datasets (see High Priority Need 2). Part of integration is
engagement with end users to make sure the approach is “fit for purpose”, easily adoptable, and scalable (see
High Priority Needs 3 &amp; 6). There must be avenues to pass on knowledge and methods about how to use these
different technologies – workshops, “visiting preceptorships”, “hubs” where people can go to learn (see High
Priority Needs 4 &amp; 7).</p>
<p>RECOMMENDATION 1. Prioritize the development and use of combinatorial NAMs. Specific NIH
activities to pursue include:</p>
<p>1.1.  Establish benchmarks and standards for individual NAMs and combinatorial NAMs to foster</p>
<p>technology integration efforts and demonstrate impact of combinatorial effect (see High Priority
Need 2: Interoperable, Reliable Datasets).</p>
<p>1.2.  Support research comparing and benchmarking relevant animal, NAMs, and human models to
validate translational potential, reduce reliance on singular methods, reduce costs, and identify
integration frameworks and strengths and weaknesses in model approaches.</p>
<p>1.3.</p>
<p>Initiate a combinatorial technology pilot for developing process/feasibility of complementing or
reducing reliance on a current animal model (i.e., nonhuman primate) for an experimental
area/condition.</p>
<p>1.4.  Track NIH’s investment in NAMs, including combinatorial NAMs, to identify gaps, support new</p>
<p>initiatives, identify promising areas for continued investment, and bolster scientific/proposal
review for tools and resources.</p>
<p>17</p>
<p>HIGH PRIORITY NEED 2: INTEROPERABLE, RELIABLE DATASETS</p>
<p>Opportunity/Need: Researchers seeking to appropriately validate the performance of NAMs need access to
reliable and interoperable high-quality datasets (including accompanying metadata), generated using both
traditional and NAM approaches. Such datasets enable qualification and validation of new methods against
benchmarks and established methods to inform the degree to which NAMs are complementary or
advantageous to traditionally employed models. For example, once an in silico method has been validated, it
can be deployed to test hypotheses generated using other datasets. Access to reliable, high-quality datasets
increases the overall efficiency, reproducibility, and validity of comparisons critical for method development.
Interoperable datasets facilitate collaboration and generation of meta-analyses from multiple studies to
provide a more comprehensive understanding of the effectiveness and limitations of the methodologies.
Additionally, validated NAMs can generate datasets that include measurements, simulations of drug response
and biomarker expressions and can generate predictions that can be further analyzed.</p>
<p>Due to the breadth of methodologies that fall under the category of NAMs and the wide swath of fields and
sectors that develop and use them, strategies are needed for promoting FAIR (findable, accessible,
interoperable, and reusable) data to foster linkage. There is a need for standards for data/metadata collection
to leverage heterogeneous NAM types and maximize data sharing and reuse across the biomedical research
enterprise. Increased dataset interoperability and reliability can only catalyze the use and development of
NAMs if the data is accessible and easy to find. To achieve this, there is a need for improved structuring and
curation for standards-based harmonization across studies.</p>
<p>Importantly, researchers and institutions often lack the infrastructure for establishing robust data sharing
practices to promote reuse of experimental data; this is further complicated by limited funding and resources
for data sharing, career pressures to retain exclusive access, or the lack of expertise needed to make datasets
reusable or interoperable. As a leader in data science and the world’s largest public funder of biomedical
research, NIH can serve as a forerunner in addressing these challenges, as it has done with its recent NIH Data
Management and Sharing Policy22 and data sharing resources created through the National Library of
Medicine (NLM) and the Office of Data Science Strategy (ODSS). Additionally, the NIH Institutes and Centers
hold the requisite expertise needed to identify data, confirm their quality, and facilitate sharing of data sets
and tools needed to qualify and benchmark NAMs. There are unprecedented collections of data and large
cohort databases containing data from animal and clinical studies, in vivo studies, environmental exposure,
genetic and -omics data, and more. Many global organizations and federal agencies have ongoing efforts in
this space, so connecting NIH’s efforts with those of these partners will be key.</p>
<p>22 https://grants.nih.gov/grants/guide/notice-files/NOT-OD-21-013.html</p>
<p>18</p>
<p>RECOMMENDATION 2. Establish resources, infrastructure, and collaborations to promote the use
of interoperable, reliable, and well curated/high quality datasets produced from research using NAMs.
Specific NIH activities to pursue include:</p>
<p>2.1. Define and address barriers to creating shared, reliable, and interoperable datasets, including
heterogeneous data/metadata terminology, formats, and standards; inconsistent governance
practices and quality curation; and lack of qualified personnel for effective communication,
translation, and adoption. Develop and disseminate methods for assessing the quality of NAM
data, leveraging existing data quality metrics.</p>
<p>2.2. Establish and maintain NAM data management policies and infrastructure to facilitate</p>
<p>heterogenous NAM data sharing and integration, including:</p>
<p>o  creation of registries, harmonization of nomenclature, development of ontologies, etc.
o  development of policies to promote timely and accessible publication of studies with</p>
<p>concurrent deposition of related datasets, assigned codes, and algorithms.</p>
<p>2.3. Identify or establish a designated repository for NAM data sharing, consistent with FAIR principles,
privacy protections, and security practices, with sufficient metadata requirements to promote
equitable reuse of high-quality NAMs data.</p>
<p>2.4. Create alliances and collaborations for collecting, managing, sharing, and publishing high-quality</p>
<p>NAMs data, including increasing access to hard-to-access data such as:</p>
<p>Industry data, focusing on the pre-competitive space and regulatory approval submissions.</p>
<p>o
o  Unpublished data, particularly from failed studies (in an effort to address survival bias).
2.5  Crowdsource new methodologies that enable access to quality data to use for qualification or</p>
<p>generated by NAM research, to improve characterization of data and increase confidence in NAM-
generated data.</p>
<p>19</p>
<p>HIGH PRIORITY NEED 3: EFFECTIVE TECHNOLOGY DISSEMINATION AND INTERCONNECTION</p>
<p>For effective technology dissemination, developers of NAMs need to consider the users of these models
starting from ideation, including the academic and industry partners who will be using them, the regulators
who will be evaluating them, and the communities who will both be informing and will be informed by them.
NAMs developers must have a deep understanding of the end user and what they need, whether it be a
physician, a patient, a researcher, or a pharmaceutical developer.</p>
<p>Rapid dissemination of reliable technology across the research community requires clarity regarding
technology “maturity” for use and dissemination. Clear descriptions of the methods are needed to ensure the
reliability and availability of resources (e.g., tools, reagents, cells, algorithms, and natural language processing
and machine learning code) to create a standard (i.e., major source of variability with in vitro methods is with
the cell sources). There is also a need for a process for verifying and validating the integrated technologies as a
whole rather than individually. In these efforts, the community must be cognizant that a failure does not mean
that every component is invalid, but rather that the components may need to be integrated in a different way
or that certain components may need to be eliminated, supplemented, or substituted out for others.</p>
<p>RECOMMENDATION 3. Promote effective dissemination and interconnection of NAMs
technologies. Specific NIH activities to pursue include:</p>
<p>3.1. Establish mechanisms to support testing, validation, qualification, and benchmarking of</p>
<p>integrated systems to maximize uptake of these systems by the community, including
frameworks for describing which stakeholder should advance which component.</p>
<p>3.2. Create accessible and reliable sources and repositories for disseminating validated NAMs.</p>
<p>o</p>
<p>Integrate strategies for deploying technologies broadly and equitably, including to under-
resourced organizations and research areas.</p>
<p>o  Create and expand access to donated human tissue repositories including both typical and</p>
<p>atypical/disease samples of tissues and cells.</p>
<p>3.3. For integrated NAMs, incentivize research focused on making the technology simpler, faster, and
cheaper (e.g, automation, miniaturization) and promote accessibility through easily navigable
licensure procedures to manage intellectual property, commercial applications, and use issues.</p>
<p>3.4. Define expectations for NAM studies to follow established reporting guidelines for funders and
publishers regarding NAMs development and use (e.g., RIVER (Reporting In Vitro Experiments)).</p>
<p>To move these technologies into widespread use, there must be targeted efforts to broaden technology reach
beyond the experimental; in other words, the emphasis should be on development of tools for the sake of
understanding human physiology rather than on the development of tools for the sake of developing new
tools. The challenge that the tools will address needs to be well defined. Platforms and technologies should be
designed to be “fit for purpose”, i.e, sufficiently complex only to the point of what is needed to answer the
biological question. All of this will require strategies for long-term investment to move technology from the
bench to broader use and will benefit from collaborations with experts in user-centered design.</p>
<p>20</p>
<p>HIGH PRIORITY NEED 4: COMPREHENSIVE TRAINING</p>
<p>New and evolving tools and technological capabilities should be disseminated to a wide scientific user base,
along with the knowledge required to wield them. The dissemination of skills to use these technologies across
the research community is essential for unleashing the catalytic possibilities with the use of NAMs. To enable
the broadest possible impact of newly developed methods and their rigorous application, support should be
provided for comprehensive training. This approach requires targeted strategies across career stage and role
in the biomedical research ecosystem. Any proposed training efforts must center around equity and
accessibility to technological approaches and training from early on in the development process, including
considerations for lower resourced areas.</p>
<p>Well-designed training courses have the potential to promote interdisciplinary and collaborative actions as
well as to provide the backbone for the cultural change required to actuate the use of novel tools and
technologies. Distributed workforce training programs can utilize and bolster industry and academic
partnerships, while fellowship programs can boost early-career cross-sector training. Opportunities to develop
“train the trainer” programs can further aid in the development and expansion of use of these methods and
increase the confidence in the results of these methods. Integrative cross -technologies and -scientific experts,
who have a deep understanding and proficiency in multiple distinct technology or scientific fields and possess
the ability to integrate knowledge and principles from these domains to solve complex problems or create
innovative solutions, are needed to bridge the gaps between various disciplines and leverage their
interdisciplinary expertise to develop holistic and synergistic approaches that require the integration of
technology from diverse fields. Similarly, technology translators, who possess the ability to facilitate effective
communication and collaboration among diverse stakeholders in a multidisciplinary environment, also play a
crucial role in enhancing interdisciplinary collaboration, promoting innovation, and ultimately improving
health outcomes by facilitating effective communication and mutual understanding among experts from
diverse backgrounds. Both integrative cross -technologies and -scientific experts and technology translators
are crucial for enhancing interdisciplinarity of training sessions while facilitating effective communication and
mutual understanding of NAM technologies among trainers and trainees from diverse backgrounds.
Different training will be necessary for the development and use of different types of NAMs. For example,
training in cyberoperations is necessary to support the use of NAMs for research and deployment utilizing
patient data. Considerations of patient privacy, cybersecurity, and respect for participant autonomy must be
considered in regard to data handling, data analysis, and safety and security of hardware devices, especially
those that become part of a closed-loop system. Importantly, for NAMs to be successfully used and deployed,
training is needed across the NAMs pipeline from development and deployment to scientific review. This
means that training is also needed for reviewers in terms of understanding proposals and the unique value of
NAMs.</p>
<p>21</p>
<p>RECOMMENDATION 4. Invest in comprehensive training to bolster continuous advances in NAMs
development and use. Specific NIH activities to pursue include:</p>
<p>4.1. Incentivize cross-training opportunities across scientific disciplines, animal to human approaches,</p>
<p>and technologies, including across sectors.
o</p>
<p>Initiate mechanisms to support multiple aspects of NAMs-based research, especially the
frontier of merging abiotic and biotic NAMs and combinatorial expertise across traditional
models.</p>
<p>o</p>
<p>Establish trainings in responsible data management and sharing unique to NAMs data types
to foster integration.</p>
<p>o  Promote training for grant reviewers to better understand how to evaluate the use of NAMs</p>
<p>in fundamental and applied research grants.</p>
<p>4.2. Create funding mechanisms for technology developers to both receive and advance training in</p>
<p>different methods and strategies for reliable technology deployment.</p>
<p>4.3. Invest in training across the research to implementation pipeline, including addressing hurdles in</p>
<p>bringing technologies to fruition, such as regulatory and policy requirements, patient care, etc.
For example:
o</p>
<p>Embed academic researchers in industry, regulatory agencies (e.g. FDA or EPA), national
laboratories, and policy not-for-profits (e.g. AAAS, PCRM).</p>
<p>o  Create collaborations between researchers and clinicians to incorporate patient perspectives</p>
<p>in NAMs development.</p>
<p>o</p>
<p>Foster entrepreneurship training.</p>
<p>4.4. Promote awareness and understanding of NAMs through publicly available educational course</p>
<p>modules and workshops covering the lifecycle of NAMs, from conceptualization to dissemination,
use, and commercialization.</p>
<p>HIGH PRIORITY NEED 5: MULTIDISCIPLINARY TEAMS</p>
<p>The full potential of NAMs can be realized through collaboration across multiple sectors and disciplines and by
integrating cross-disciplinary research with its implementation. This is an endeavor where biologists,
biomedical engineers, data scientists, integrative/cross technologies experts and many other experts should
engage in a team-science approach to create innovative solutions in catalyzing NAMs with the highest
relevance to human biology. Moreover, the progression from traditional to newly evolving methodologies is
not only about technological advancements, but it also reflects a broader shift in the biomedical landscape
influenced by economic, regulatory, and public sentiments. The scientific relevance and cost-effectiveness of
new methods, combined with regulatory needs and incentives, have accelerated the momentum toward
developing innovative research approaches. Additionally, the fusion of diverse fields from epidemiology,
genetics, cell biology, and immunology with engineering (i.e., “biological engineering”) and machine learning
with biomedical research is enabling a more holistic approach, redefining the boundaries of what is possible.
This interdisciplinary synergy is central to the promise of NAMs, underscoring the importance of collaboration
in pioneering solutions that could increase understanding of human biology and how best to model or mimic
the human condition. Amidst this transformation, the scientific community stands on the brink of a new era,</p>
<p>22</p>
<p>driven not just by the quest for knowledge but by the aspiration to reshape the very fabric of how biomedical
research is conducted.</p>
<p>To achieve the full potential of collaborative interdisciplinary efforts, there is a need to engage along the
lifecycle of NAMs development with teams that have specialty expertise, in addition to those who work with
more traditional models and who would ultimately apply the research findings to human patient cohorts. This
includes regulatory expertise and connections regarding future evaluation of technologies as well as expertise
in community and patient engagement (see High Priority Need 6: Socially Responsible Technologies). There
must be incentives to bring together academics, technology providers, and industry partners in a pre-
competitive, consortium-type environment. Each sector comes with its own needs and different levels and
types of resources. The success of multidisciplinary teams relies upon the communication between disciplines
and will require 1) standardized language to speak across specialties and sectors; 2)  professionals who
possess extensive knowledge and expertise in multiple scientific and technological disciplines; 3) individuals
who possesses the unique ability to facilitate effective communication and collaboration among diverse
stakeholders in a multidisciplinary environment; and 4) increased data sharing across sectors.</p>
<p>RECOMMENDATION 5. Facilitate multidisciplinary teams with expertise across technologies and
the lifecycle of NAMs development and use. Specific NIH activities to pursue include:</p>
<p>5.1. Develop funding opportunities to support multi-disciplinary teams, considering potential:</p>
<p>o  Scientific, technological, and engineering needs
o  Regulatory or policy requirements
o  Ethical considerations
o  Patient/public adoption</p>
<p>5.2. Support incentives for multi-laboratory coordination, especially mechanisms for supporting</p>
<p>expertise across the lifecycle of development and use.</p>
<p>5.3. Create novel funding opportunities such as cross-disciplinary challenge programs or prize</p>
<p>competitions.</p>
<p>5.4. Promote annual conferences or symposia that mobilize varying perspectives and expertise and
establish resources and mechanisms to assist researchers in connecting with experts across
disciplines, sectors, and research stages.</p>
<p>5.5. Support pilot studies incorporating multidisciplinary expertise focused on studying the
predictivity of certain models before publishing draft policies and risk assessments.</p>
<p>HIGH PRIORITY NEED 6: SOCIALLY RESPONSIBLE TECHNOLOGIES</p>
<p>Integrating societal norms into the development and use of emerging technology is a mainstay of building
trust and promoting uptake of technological advancement. This is a long-studied field, and there is immense
value in learning from these experts to ensure new technologies do not fall into the traps of the past.
Additionally, those developing and using NAMs should work in tandem with bioethicists, patients, and
policymakers to create and sustain a responsible culture of use. Again, not unique to NAMs, but there are
considerations that researchers should build into their studies from conception through deployment.</p>
<p>23</p>
<p>In creating models that recapitulate human biology, researchers should prioritize the autonomy, privacy, and
needs of the individual(s) sharing the source data or biospecimens. For example, cells used for in vitro
methods originate from donor tissues; thus issues of consent, identification, return of value, etc., need to be
discussed and addressed early in the conceptualization of the model. Additionally, while use of human
biospecimens can capture unique aspects of human biology, they still fail to represent the diversity of humans
in general. In silico methods offer tremendous promise for complementing in vitro and in chemico methods in
this domain. However, considerations around algorithmic biases in collection, analysis, and interpretation, as
well as the cultural considerations and potential for stigmatization, need to be considered. Recommendations
made throughout this report, including standards and documentation in the development of these models,
are a first step in addressing these issues.</p>
<p>In the development and use of any emerging or evolving technology, there will be required tradeoffs between
access, cost, and consistency/reproducibility. How these factors are individually weighed will greatly influence
the best course for moving forward, especially in terms of equity and accessibility. There are also tradeoffs
regarding commercialization incentives and the use of incentive structures or subsidies may be required
especially early on in development to promote access. Critical consideration must be given to striking a
balance where actions around intellectual property can incentivize commercialization instead of creating
unwarranted barriers to entry. Deciding whether and when intellectual property is warranted to advance
broader goals is often a fact-dependent question, and there are not likely to be bright-line rules around using
intellectual property rights to promote socially responsible creation and deployment of NAMs.</p>
<p>RECOMMENDATION 6. Promote social responsibility in both the creation and deployment of NAMs
across the research lifecycle. Specific NIH activities to pursue include:</p>
<p>6.1. Foster equitable development and use of NAMs for research and public benefit. For example:
o  Support research to characterize unique attributes of NAMs to understanding individual</p>
<p>differences, method biases, etc., and to recognize, minimize, and correct for variability and
biases.</p>
<p>o  Promulgate guidance for considering sources of tissues, cells, and data/metadata used to
develop NAMs and whether they are sourced ethically or represent population diversity.</p>
<p>o  Promote open sharing of technology and data when possible.</p>
<p>6.2. Strengthen interagency partnerships to develop a coordinated federal approach to NAMs that</p>
<p>enables science to advance efficiently, safely, and ethically while minimizing administrative and
regulatory burden.</p>
<p>6.3. Support cost effective analyses of proposed technologies with existing methods, including</p>
<p>animal studies, looking at time, scalability, and resource efficiency.</p>
<p>6.4. Support bioethical research on ethical, legal, and social issues unique to NAMs, including</p>
<p>research to maximize responsible deployment, promote equity, and provide “return of value”
to research participants and communities.</p>
<p>6.5. Partner research initiatives with robust public engagement to incorporate social norms and</p>
<p>promote awareness of emerging technologies.</p>
<p>Importantly, maintaining scientific integrity and public trust requires honest, transparent, and balanced
discussion regarding the opportunities and limitations of NAMs. To be clear, there will bethe Working Group
expect positive return on the public’s investment in NAMs, but there is risk in overpromising technological</p>
<p>24</p>
<p>capabilities that damage confidence in the biomedical research enterprise more broadly. The research
community has the responsibility to the public to be open and honest in terms of communicating the NAMs
agenda.</p>
<p>HIGH PRIORITY NEED 7: COORDINATED INFRASTRUCTURE</p>
<p>Coordinated infrastructure is required for addressing virtually all the high priority needs above. In order to
meet the full potential of NAMs to help researchers uncover new discoveries about human health and disease,
there must be dedicated resources and venues to support these activities.</p>
<p>These investments should prioritize venues for strengthening dialogue across and even within sectors and
geographical boundaries. With rapid development of any technology, infrastructure is needed that can
minimize duplication of effort and maximize taxpayer funding with equity as a priority.</p>
<p>RECOMMENDATION 7. Support and maintain coordinated infrastructure to catalyze effective and
responsible NAM development and use. Specific NIH activities to pursue include:</p>
<p>7.1. Create mechanisms for disseminating NAMs resources, technologies, and expertise efficiently,</p>
<p>equitably, and reliably across researchers and institutions. For example:
o  Protocols for technology development and use, qualification of reagents and equipment, tracking</p>
<p>of materials and experimental details, and standard operating procedures for teams.</p>
<p>o  Clearing houses and repositories for easy, reliable, and inexpensive access to specialty reagents</p>
<p>and custom syntheses.</p>
<p>o  Knowledge bases for tracking NAMs, how they are used, for what purposes, and how in</p>
<p>combination with other models.</p>
<p>7.2. Promote or establish consortia and venues for sharing established best practices, standards,</p>
<p>definitions, frameworks, and harmonized approaches for NAMs.</p>
<p>7.3. Invest in infrastructure to support institutions in keeping pace with the rapid evolution of NAMs,</p>
<p>including:
o  Establishing "recruitment and placement" platforms and collectives so that researchers can</p>
<p>identify colleagues with specialized expertise.</p>
<p>o  Supporting small
o   to mid-scale physical laboratory infrastructure.</p>
<p>7.4. Identify opportunities to build upon existing efforts both nationally and internationally to link</p>
<p>resources and identify a clear source of coordination for NAMs resources.</p>
<p>7.5. Establish dedicated and centralized core facilities as national or regional resources to develop and</p>
<p>run NAM assays to reduce costs, leverage scale, and provide training.</p>
<p>25</p>
<p>SECTION IV. SUCCESSFUL INTEGRATION: TRANSFORMATIVE PROJECTS</p>
<p>TRANSFORMATIVE PROJECT EXEMPLAR 1: LEVERAGING IN VITRO AND IN SILICO
TECHNIQUES FOR ACTIONABLE INSIGHTS IN COMPLEX BIOLOGICAL MODELS</p>
<p>Human Applicability. The process of cell culture is intricate, particularly with the advent of new cell lines and
the sourcing of specific lines from patients. Optimizing cell culture is vital for ensuring reproducibility and
reliability. However, traditional methods are often time-consuming and resource-intensive, especially when
dealing with a limited number of cells, such as those from individual patients. Assessing cell morphology, a
crucial aspect of this process, is influenced by numerous factors, making it both time-consuming and
subjective. Furthermore, as in vitro assays grow in complexity, there is a notable absence of standard tools or
metrics for evaluating performance or biological function within these models. It is important to note that
commonly used morphological metrics may not accurately reflect the biological function of the network, such
as oxygen transport.23</p>
<p>Challenges for Traditional Models. The challenge extends to assessing drug responses within organoids or
microphysiological systems, especially when handling high-throughput image datasets, which is both difficult
and labor-intensive. Omics-based approaches are increasingly employed to identify disease mechanisms and
drug responses, as well as to detect adverse effects. However, the insights derived from omics-based data in in
vitro assays do not always yield valuable inferences.24</p>
<p>Combinatorial Approach. Combining in vitro and in silico techniques offers a scalable approach that aligns
with the objectives and constraints of cell culture processes. For instance, machine learning algorithms have
been effectively used to optimize cell culture feeding strategies. This optimization enhances cell growth and
metabolite production, leveraging in vitro cell culture process data to create and evaluate both linear and
nonlinear models through real-time experiments.25,26</p>
<p>Additionally, machine learning has been employed to quantify cell morphology in a scalable and cell-agnostic
manner, extending its applicability across various cell types.27,28,29 This includes the application of tools such as
random forest regression models for assessing the biological functions of microphysiological systems.30
Additionally, the fusion of simulation and predictive models with omics data from in vitro assays can help to
enhance therapeutic performance.31</p>
<p>The use of computer vision tools for analyzing high-throughput image datasets from in vitro assays enables
detailed and automated analyses. An example of this is the automated single-organoid morphology analysis in</p>
<p>23 Tronolone JJ, Mathur T, Chaftari CP, Jain A. Evaluation of the morphological and biological functions of vascularized microphysiological systems with supervised
machine learning. bioRxiv [Preprint]. 2023 Jan 15. Update in: Ann Biomed Eng. 2023 Mar 13.
24 Yue R, Dutta A. Computational systems biology in disease modeling and control, review and perspectives. NPJ Syst Biol Appl. 2022 Oct 3;8(1):37.
25 Rashedi M, Rafiei M, Demers M, Khodabandehlou H, Wang T, Tulsyan A, Undey C, Garvin C. Machine learning-based model predictive controller design for cell
culture processes. Biotechnol Bioeng. 2023 Aug;120(8):2144-2159.
26 Rashedi M, Rafiei M, Demers M, Khodabandehlou H, Wang T, Tulsyan A, Undey C, Garvin C. Machine learning-based model predictive controller design for cell
culture processes. Biotechnol Bioeng. 2023 Aug;120(8):2144-2159.
27 Welter EM, Kosyk O, Zannas AS. An open access, machine learning pipeline for high-throughput quantification of cell morphology. STAR Protoc. 2023 Mar
17;4(1):101947.
28 Sherman J, Verstandig G, Brumer Y. Application of machine learning to large in-vitro databases to identify cancer cell characteristics: telomerase reverse
transcriptase (TERT) expression. Oncogene. 2021 Aug;40(31):5038-5041.
29 Mergenthaler P, Hariharan S, Pemberton JM, Lourenco C, Penn LZ, Andrews DW (2021) Rapid 3D phenotypic analysis of neurons and organoids using data-driven
cell segmentation-free machine learning. PLoS Comput Biol 17(2): e1008630.
30 Tronolone JJ, Mathur T, Chaftari CP, Jain A. Evaluation of the morphological and biological functions of vascularized microphysiological systems with supervised
machine learning. bioRxiv [Preprint]. 2023 Jan 15. Update in: Ann Biomed Eng. 2023 Mar 13.
31 Yue R, Dutta A. Computational systems biology in disease modeling and control, review and perspectives. NPJ Syst Biol Appl. 2022 Oct 3;8(1):37.</p>
<p>26</p>
<p>chemotherapy dose-response experiments, which has identified significant dose-related effects on organoid
features like circularity, solidity, and eccentricity.32</p>
<p>These technologies are mutually beneficial: in silico methods depend on data from in vitro systems, and
conversely, in silico techniques maximize the utility of in vitro assay data. This interdependence facilitates the
design of in vivo studies and potentially reduces the reliance on preclinical in vivo models. By streamlining this
process, these combined technologies not only optimize research efficiency but also contribute to more
precise and effective biological research outcomes.</p>
<p>TRANSFORMATIVE PROJECT EXEMPLAR 2: UNCOVERING NEW UNDERSTANDING OF
NEUROPSYCHIATRIC DISORDERS</p>
<p>Human Applicability. Developmental and degenerative disorders of the human nervous system collectively
represent one of the largest causes of disability and disease burden worldwide, yet there is a limited
understanding of the mechanisms leading to disease. Moreover, currently there are relatively limited
therapeutic approaches.</p>
<p>Challenges for Traditional Models. Animal models have yet to prove their fidelity and utility for studying
psychiatric disorders, which are behaviorally defined. Primate models represent a promising avenue, but
access to genetically engineered non-human primates is often limited and developing these models often
takes many years. Human cellular models, such as organoids or assembloids, maintain the genetic background,
but the inherent variability of patient-derived cell lines and inconsistencies between organoids have been
problematic and they do not recapitulate circuit-level features or display behavioral outputs.</p>
<p>Combinatorial Approach. Concepts concerning psychiatric disorder pathophysiology and therapeutic targets
have shifted over the years. There have been strong advances in identifying the genetic architecture and
susceptibility for many of these conditions, including autism spectrum disorders, schizophrenia, epilepsy,
intellectual disability, addictions, mood disorders, Alzheimer disease, and others. There is a need for an
integrated model of the human disease to complement, reduce reliance or replace animal models, but the
pathophysiology of psychiatric disorders may impact multiple cell types including neurons, microglia,
astrocytes and possibly vascular cells, and in some cases neuro-immune interactions. Current stem cell
technologies can recreate in vitro 3D organoid models using donor cells that can recapitulate some of the
cellular diversity and complex cellular architecture of the central and peripheral nervous system and model
cell migration and circuit assembly in a genetic background of disease vulnerability. Some of these methods
are being developed at scale, in a way that may enable drug screening. Moreover, xenograft transplantation of
organoids could allow for advancing cell maturation in an in vivo environment and enable therapeutic testing
on human cells in an in vivo context. There are also significant efforts to develop multi-omics assays to capture
and perturb the genome, transcriptome or chromatin landscape, as well as functional assays that capture the
activity of neurons. Coupled with machine learning-aided taxonomy, neural organoids and assembloids may
fill the need for integrated systems via high throughput production techniques.</p>
<p>32 Matthews JM, Schuster B, Kashaf SS, Liu P, Ben-Yishay R, Ishay-Ronen D, et al. (2022) OrganoID: A versatile deep learning platform for tracking and analysis of
single-organoid dynamics. PLoS Comput Biol 18(11): e1010584.</p>
<p>27</p>
<p>TRANSFORMATIVE PROJECT EXEMPLAR 3: COMBINING IN VITRO, IN VIVO (CLINICAL AND
PRECLINICAL), AND IN SILICO APPROACHES TO MINIMIZE DEPENDENCE ON PRECLINICAL IN
VIVO MODELS</p>
<p>Human Applicability. Transdermal drug delivery, as an evolving field, presents a significant shift from
traditional methods of drug administration. This method is increasingly favored for its potential to enhance
patient compliance, particularly in those who find oral or injectable routes challenging. A key advantage of
transdermal delivery is its ability to circumvent pre-systemic metabolic effects and reducing the likelihood of
adverse effects through decreased systemic exposure.</p>
<p>Challenges for Traditional Models. Despite these advantages, there are substantial challenges in the
development and assessment of transdermal therapeutics. Currently, the evaluation of these drugs' toxicity
and efficacy relies heavily on traditional in vitro and in vivo assessments.</p>
<p>In vivo studies, particularly those utilizing animal models like rats, are a staple in assessing the safety and
efficacy of transdermal drugs. These models offer a more comprehensive understanding of how a drug
behaves in a living system, including its metabolism, distribution, and excretion. However, reliance on these
models presents several challenges. Firstly, in vivo testing is time-consuming, often requiring extended periods
to observe long-term effects of the drug. Secondly, it is costly.</p>
<p>Moreover, translating findings from animal models to humans is not always straightforward due to
physiological differences between species. This can lead to uncertainties in predicting how a drug will perform
in humans based on animal data alone.</p>
<p>In summary, while transdermal drug delivery offers several advantages over traditional routes, it also presents
unique challenges in terms of development and safety assessment.</p>
<p>Combinatorial Approach. In the realm of dermal exposure, in silico physiological modeling plays a crucial role.
It is instrumental in predicting the effects of therapeutic agents and in assessing the impact of different
formulations on transdermal disposition. This approach is becoming increasingly important as it offers a more
efficient and potentially more accurate method for evaluating drug efficacy and safety.</p>
<p>The integration of in silico predictions, rooted in in vivo clinical data, stands as a pivotal advancement in
pharmacological research, particularly in the realm of transdermal therapeutics. This approach leverages
scalable and accurate computational models, harnessing extensive datasets derived from in vivo clinical trials.
Such data encompasses a vast array of parameters and coefficients, which are instrumental in training
sophisticated Quantitative Structure-Activity Relationship (QSAR) models. These QSAR models are adept at
predicting the potential utility of compounds when administered transdermally.33 By doing so, they have the
potential to significantly reduce the reliance on traditional in vivo preclinical models.</p>
<p>In a similar vein, the application of machine learning techniques in pharmacological studies offers a
transformative approach to predicting drug toxicity. By analyzing and interpreting in vitro human data,
particularly transcriptome profiles, machine learning algorithms can make informed predictions about a
compound's toxicity.34 This approach provides valuable insights that can guide and inform subsequent in vivo
studies, thereby optimizing the research process and potentially reducing the need for extensive in vivo
testing.</p>
<p>33 Maharao N, Antontsev V, Hou H, Walsh J, Varshney J. Scalable in silico Simulation of Transdermal Drug Permeability: Application of BIOiSIM Platform. Drug Des
Devel Ther. 2020 Jun 11;14:2307-2317.
34 Gardiner LJ, Carrieri AP, Wilshaw J, Checkley S, Pyzer-Knapp EO, Krishna R. Using human in vitro transcriptome analysis to build trustworthy machine learning
models for prediction of animal drug toxicity. Sci Rep. 2020 Jun 12;10(1):9522.</p>
<p>28</p>
<p>TRANSFORMATIVE PROJECT EXEMPLAR 4: COMBINATORIAL APPROACHES TO IMPROVE
TREATMENT OF CHRONIC INFLAMMATORY CONDITIONS</p>
<p>Human Applicability. There is increasing appreciation that chronic inflammatory conditions as exemplified by
endometriosis, ME/CFS, chronic Lyme disease, and Crohns, are not “one disease” – i.e., that patients can likely
be classified into subgroups on a molecular basis, in a manner similar to cancer patients but using different
kinds of molecular markers as somatic mutations are not yet established as causative. Many of these diseases
skew strongly female, are difficult to diagnose, and lack effective, safe therapies for large fractions of the
afflicted. For example, an estimated 200 million women worldwide suffer from endometriosis, which is so
lacking in adequate therapies that many patients resort to repeated major surgical procedures to alleviate
debilitating symptoms.</p>
<p>Challenges for Traditional Models. A compendium of factors makes these conditions extremely difficult to
model in animals or with current computational models. First, for many chronic inflammatory diseases,
genetic linkage and GWAS studies have identified multiple loci associated with disease, yet individuals with
genetic signatures may be free of disease. This conundrum suggests that there may be mitigating factors, such
as undiscovered protective loci, or exposure to infection, environmental chemicals, or stress in ways that
cause epic genetic modifications across multiple loci; emerging data supports these hypotheses.  Second,
animal models fail to capture much of the human patient symptoms adequately, especially pain, fatigue, and
uterine-specific symptoms such as heavy menstrual bleeding. Pain mechanisms, especially the kinds of chronic
pain associated with inflammatory diseases, are notoriously sexually dimorphic in humans. Finally, human
patients often have multiple co-morbid conditions, and chronic inflammatory diseases often afflict multiple
organ systems. They are thus very challenging to study in the context of the traditional NIH Institute/Center
funding models, which tend to isolate organ systems (e.g. NIDDK, NHLBI) and disease triggers (e.g. NIAID),
precluding easy construction of multi-disciplinary clinical teams.
These factors together motivate NAMS that can capture the diversity of patient sub-populations and point to
precision approaches based on a compendium of knowledge that cannot be gleaned from traditional models.</p>
<p>Combinatorial Approach. Endometriosis, which afflicts about 10% of U.S. women, provides a useful example
to illustrate integration of multiple types of NAMs in chronic inflammatory diseases. New approaches are
starting to chip away at the significant individual bottlenecks that still exist in diagnostics, including in patient
symptom phenotyping, imaging, and molecular markers, though approaches thus far have focused almost
exclusively on binning patients into “disease or no disease” categories, without providing clinically actionable
solutions for definitive therapies. Several heralded molecular markers described in blood have not reached
clinical replicability or acceptance. Enhanced understanding and deployment of pain phenotyping at the
research (e.g. fMRI) and clinical levels is starting to yield some path forward to potential targeted therapies
from among those that already exist. More advanced bioinformatics analysis of genetic loci, by using multi
variate analysis of genotypes to cluster patients into subgroups according to correlated changes in multiple
SNPs, are starting to yield mechanistic insights into precision medicine approaches for patient subgroups, and
insights into overlap with other diseases that are co-morbid such as inflammatory bowel disease. Translating
sets of correlated SNPs into testable hypotheses involving potential therapeutic targets requires new
approaches in human systems modeling, as the animal correlates are lacking. There is growing appreciation of
the dire need for sexually dimorphic systems models for how inflammation changes across the lifespan, from
pre-puberty, through puberty, and through later life stages, especially as endometriosis is often erroneously
believed to be a disease of women in their twenties through late 40s, when it actually afflicts a significant
fraction of women in their early teens and postmenopausally. Beyond genomics, a growing compendium of
multi-omic data sets on lesion properties and on dysregulation of other affected organs is yielding insights into</p>
<p>29</p>
<p>immunological factors operative at the sites of local inflammation. However, unknown is how more systemic
immunological factors, including potential immune system rewiring from prior infection, contribute to overall
immunological response, as has been demonstrated by the causative role of EBV infection in triggering
multiple sclerosis. These types of analyses, that promise to yield critical insights into the critical
pathophysiological factors distinguishing different patient subgroups from each other, are crucial for the
design of MPS models to capture disease phenotype in precision fashion.</p>
<p>The design and use of MPS for chronic inflammatory diseases is still in its infancy, in part because the
aforementioned design principles are still scant, and in part many of the essential in vitro tools for sexually
dimorphic immunologically-competent MPS are yet to be developed or are far from being deployed at scale.
For example, control of steroid hormone microenvironment requires careful attention to the equilibrium
between hormones and plasma proteins, along with metabolism; this can be especially challenging in the most
commonly-used MPS formats, which absorb steroids. Most inflammation processes involve complex
interactions between immune cells and multiple cell types in tissues. Scaling MPS models to accurately
represent these processes requires integration with computational models of cell-cell crosstalk in a dynamic
microenvironment, and design of MPS hardware that enables appropriate acquisition of information
noninvasively (eg imaging, sampling media) or at endpoints, recognizing that each MPS replicate may be
expensive to produce.  Finally, computational modeling of systems pathophysiology can point to design of
interacting MPS modules to represent organ-organ crosstalk. For example, in endometriosis, some patients
may have a gut-liver-uterus-neuroimmune axis. While modeling neuroimmune – tissue interactions in MPS is
still a nascent field, it is integral to understanding complex inflammatory diseases where pain is a prominent
phenotype.</p>
<p>TRANSFORMATIVE PROJECT EXEMPLAR 5: DATA INTEGRATION ACROSS NAMS, TRADITIONAL
MODELS, AND THE CLINIC TO EMULATE PATIENT-SPECIFIC TUMOR-IMMUNE
ENVIRONMENTS</p>
<p>Human Applicability. Current projections show that 40% of population will be diagnosed with cancer within
their lifetime. Despite major advances in cancer diagnosis and therapy, many patients do not respond or
relapse after treatment. Metastatic progression of circulating cancer cells to distant organs remains the major
determinant of poor outcome, being the cause of 90% of all cancer deaths. While decades of cancer biology
research have focused on oncogenic transformations leading to the emergence of primary tumors, much less
attention has been directed to studying how tumor cells alter their microenvironment and colonize a distant
organ, and what determines their dormancy and activation. This gap is largely due to the lack of adequate
experimental models.</p>
<p>Metastatic progression is particularly difficult to study, both in patients and in the existing models. Cell culture
and animal models remain poor predictors of human pathophysiology of cancer, its metastatic dissemination
and response to treatment. Two additional aspects are contributing to the incredible complexity of cancer: (i)
biological heterogeneity of cancer cells and their microenvironments among the cancer patients and even
within the same patient, and (ii) cancer cell interactions with the immune system. Both the animal models and
cultures of cancer cell lines largely fail to recapitulate these critical aspects. The development of predictive
human tissue/organ models of cancer metastasis that can recapitulate key aspects of cancer pathophysiology
would be transformative to cancer research and pre-clinical validation of new therapeutic modalities,
representing a real opportunity for integrated NAMs to address the major scientific and clinical challenges of
cancer research and therapy.</p>
<p>30</p>
<p>Challenges for Traditional Models. Metastatic dissemination involves a complex series of events including
extravasation from vasculature, homing to target tissue, and environmental adaptation fostering interactions
with resident cells and the target-specific extracellular matrix with distinct biophysical and biochemical
foundations that are different of those at the site of origin. Metastatic cells from primary tumors tend to
assume new and rapidly evolving molecular programs as they colonize the new organ environments. The
complexity of these processes that is further compounded by the spatiotemporal heterogeneity of cancer cells
and their interactions with immune cells. These aspects of metastatic progression are particularly difficult to
model in cultures of cancer cell lines and experimental animals, requiring human tissue contexts and
individualized, patient-specific contexts. Appropriate models of cancer metastasis would maintain the tumor
cells in a low-proliferative state with the display the metastatic phenotype associated with drug resistance.
The models of interest would provide biologically meaningful and tightly controllable environments designed
to elucidate mechanistic drivers of metastasis and therapeutic predictions using NAMs validated against
matched patient outcomes. Integrated NAMs designed to recapitulate the key steps of metastatic
dissemination will be critical for advancing understanding of biological mechanisms governing metastatic
progression and paving the way for development of drugs for late-stage disease.</p>
<p>Combinatorial Approach. The studies of cancer progression, metastasis and responses (or resistance) to drugs
or immunotherapy are an area that would critically benefit from the patient-specific NAMs and the integrated
NAMs approaches. A recent example of an in vitro NAM of this kind is the multi-organ chip with matured
tissue niches linked by vascular flow. Each tissue is cultured in its own optimized environment and is separated
from the common vascular flow by a selectively permeable endothelial barrier. This design allows the
interlinked tissues to maintain their molecular, structural and functional phenotypes over long culture times
while enabling tissue cross talk by molecular species, extracellular vesicles and circulating cells. The model
recapitulated the pharmacokinetic and pharmacodynamic profiles of doxorubicin observed clinically, and
allowed for the identification of early miRNA biomarkers of cardiotoxicity. One can envision that NAMs of this
kind, configured by linking to each other the human tissues representing cognate metastatic sites (e.g., bone,
brain, liver, lung) engineered from iPSCs could be used to study metastasis of unprocessed tumor cells
obtained from a patient along with the matching immune cells and introduced into the vascular circulation.
The experimental studies in such organs-on-chip platforms can be further supported by advanced imaging and
gene editing (to track cells and their functions), spatial genomics and computational modeling. Integration of
these studies with the in silico and in chemico NAMs and benchmarking against clinical data and relevant
animal studies offer tremendous potential for bringing us closer to emulating the patient-specific
architectural, cellular, and phenotypic features of tumor-immune environments and cancer cell extravasation
into diverse tissues.</p>
<p>31</p>
<p>SECTION V. CONCLUDING REMARKS
The synthesis of understanding and innovation across models and fields is no small feat. Ideally, researchers
seek to design an innovation-implementation cycle whereby researchers iterate on work done across models
to inform human health, generating data from which the cycle repeats. For NAMs, the real opportunity to
integrate AI, computational modeling, and 3-D organoids, human genomics, and more, into an increasingly
sophisticated model provides a powerful opportunity to maximize the public’s investment in biomedical
research. Ultimately, this fuels the NIH in achieving its mission by providing researchers with the diversity of
tools needed to answer complex questions about human health and disease.</p>
<p>Solving the complexities of disease will most certainly require creative and effective mechanisms for catalyzing
partnerships across the biological and biomedical research enterprise. The recommendations described
throughout this report stress the importance of uniting diverse, multi-disciplinary teams that include not only
the researchers, but a vast number of those treating, caring for, and afflicted by human disease. Breaking
down silos shall not only require setting up collaborations between groups, but also training scientists in a
multi-disciplinary fashion, creating standardized language to speak across specialties and sectors, and building
and maintaining an infrastructure so as to be able to foster data interoperability and integrated models.
Additionally, the large-scale uptake of any new technology will need to be supported by culture change, in
which needs of the groups are addressed. Enhancing communication through all stages of NAM development
and application will be essential to building trust and incorporation of the work we do.</p>
<p>Finally, at the end of the day, the biomedical research enterprise is working to advance the human condition.
Here is where our Working Group found the most hope and promise of spurring advances in NAMs. Whether
it be to diversify our research toolkit or enable new inquiries into underserved areas of disease, we firmly
believe NAMs will transform our collective capabilities to promote health. We thank the NIH for the
willingness to work with the community to chart a bold, ambitious, and equitable path forward to achieve this
important goal.</p>
<p>32</p>
<p>APPENDICES</p>
<p>APPENDIX A – Definitions and Terms</p>
<p>NAM APPROACHES
In chemico methods: experiments performed on biological molecules, such as proteins and DNA, outside of
cells, which may be used to study how these molecules interact with each other and with drugs. Specific in
chemico terms:</p>
<p>•  Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR):  a biotechnology that allows
precise editing of the genome through use of guide RNA and an endonuclease that cleaves DNA.35</p>
<p>•  Direct Peptide Reactivity Assay: a common in chemico model used in toxicology in which the
interaction of chemicals with proteins is used to predict whether they will react with skin.36</p>
<p>In silico methods: experiments performed by computing platform or custom hardware, encompassing
mathematical modeling and simulation, machine learning, and other computational techniques. Specific in
silico terms:</p>
<p>•  Computational model: a general term describing the use of computers to simulate complex systems37</p>
<p>•  Machine learning: often used synonymously with artificial intelligence (AI/ML), an approach in which</p>
<p>computers analyze and make predictions on data38</p>
<p>•  Artificial intelligence: often used synonymously with machine learning (AI/ML), a computer system
with the ability to learn how to perform tasks rather than carry out programmed instructions39</p>
<p>•  Natural language processing: a term for computer algorithms that automate the processing of</p>
<p>unstructured language, both extracting meaning from it and generating it40</p>
<p>In vitro methods: Experiments performed on cells outside of the body, including various types of cell, organoid and
tissue culture techniques. Specific in vitro terms:</p>
<p>•  Microphysiological systems (MPS): an in vitro platform composed of cells; explants derived from</p>
<p>tissues/organs; and/or organoid cell formations of human or animal origin in a micro-environment that
provides and supports biochemical/electrical/mechanical responses to model a set of specific
properties that define organ or tissue function.41</p>
<p>•</p>
<p>Tissue chip: also called organ chip or organ-on-chip, a form of MPS in which cells are grown as a three-
dimensional structure within a microfluidic chip.</p>
<p>42</p>
<p>35 Redman M, King A, Watson C, et al. What is CRISPR/Cas9? Archives of Disease in Childhood - Education and Practice 2016;101:213-215.
36 Jon F. Lalko, Ian Kimber, G. Frank Gerberick, Leslie M. Foertsch, Anne Marie Api, Rebecca J. Dearman, The Direct Peptide Reactivity Assay: Selectivity of Chemical
Respiratory Allergens, Toxicological Sciences, Volume 129, Issue 2, October 2012, Pages 421–431,
37 nibib.nih.gov/science-education/science-topics/computational-modeling
38 commonfund.nih.gov/bridge2ai
39 nibib.nih.gov/science-education/science-topics/artificial-intelligence-ai
40 nlm.nih.gov/research/focus/Natural.html
41 ntp.niehs.nih.gov/whatwestudy/niceatm/test-method-evaluations/mps
42 ncats.nih.gov/tissuechip/about</p>
<p>33</p>
<p>•  Organoids: in vitro-generated cellular systems that emerge by self-organization, include multiple cell
types, and exhibit some cytoarchitectural and functional features reminiscent of an organ or organ
region. Organoids can be generated as 3D cultures or by a combination of 3D and 2D approaches (also
known as 2.5D) that can develop and mature over long periods of time (months to years). For the
nervous system, they are generally constructed from pluripotent stem cells but can also be derived
from donor tissues with growth potential (such as glioblastoma organoids).43</p>
<p>•  Assembloids: self-organizing cellular systems resulting from the combination of a type of organoids</p>
<p>with another type of organoids (for example, dorsal forebrain with ventral forebrain) or with different
specialized cell types (for example, cortical organoid with endothelial cells) that result in integration.</p>
<p>TECHNOLOGY VALIDATION, QUALIFICATION, AND BENCHMARKING</p>
<p>•  Benchmarking: rigorous comparison of the performance of different technologies or approaches to
determine the strengths of each or to provide recommendations regarding their suitability for the
purpose at hand.44 Benchmarking has several benefits, including increasing awareness of emerging
technologies and approaches (NIH, 2023) and understanding how the performance of a new approach
methodology compares with that of in vivo approaches. Benchmarking is also appropriate for
comparing new approach methodologies relative to the same or different intended uses for a model,
allowing for identification of the most appropriate technologies and approaches for specific COUs.45,46</p>
<p>•  Validation:</p>
<p>o  Analytical validation: evaluation of data processing algorithms that convert technology-</p>
<p>collected measurements into outputted metrics.47</p>
<p>o  Clinical validation: demonstrating that technology adequately identifies, measures, or predicts
a meaningful clinical, biological, physical, functional state or experience in the specified (1)
animal cohort and (2) context of use.48</p>
<p>o  Technology validation: demonstrating that a particular method is reliable and relevant in a</p>
<p>specific research application.49</p>
<p>•  Verification:  ensuring, through demonstration of precision, reliability and reproducibility, that a</p>
<p>device is measuring and storing data accurately.50</p>
<p>•  Qualification: process used by the FDA, by which an alternative method is demonstrated to have
sufficient reliability and rigor in a specific context of use to be applied in drug development.51</p>
<p>43 Pașca, S.P., Arlotta, P., Bateup, H.S. et al. A nomenclature consensus for nervous system organoids and assembloids. Nature 609, 907–910 (2022).
44 Weber, L.M., Saelens, W., Cannoodt, R. et al. Essential guidelines for computational method benchmarking. Genome Biol 20, 125 (2019).
45 Mangul, S., Martin, L.S., Hill, B.L. et al. Systematic benchmarking of omics computational tools. Nat Commun 10, 1393 (2019). 4
46 Wu, Yi MD; Li, Shizhen MD; Yuan, Jingxiong MD; Zhang, Hang MD; Wang, Min MD; Zhang, Zhenxiong MD; Qin, Renyi MD, PhD. Benchmarking: a novel measuring
tool for outcome comparisons in surgery. International Journal of Surgery 109(3):p 419-428, March 2023.
47 Goldsack, J.C., Coravos, A., Bakker, J.P. et al. Verification, analytical validation, and clinical validation (V3): the foundation of determining fit-for-purpose for
Biometric Monitoring Technologies (BioMeTs). npj Digit. Med. 3, 55 (2020).
48 Goldsack, J.C., Coravos, A., Bakker, J.P. et al. Verification, analytical validation, and clinical validation (V3): the foundation of determining fit-for-purpose for
Biometric Monitoring Technologies (BioMeTs). npj Digit. Med. 3, 55 (2020).
49 ntp.niehs.nih.gov/whatwestudy/niceatm/iccvam/international-partnerships/icatm
50 Baran SW, Bratcher N, Dennis J, Gaburro S, Karlsson EM, Maguire S, Makidon P, Noldus LPJJ, Potier Y, Rosati G, Ruiter M, Schaevitz L, Sweeney P, LaFollette MR.
Emerging Role of Translational Digital Biomarkers Within Home Cage Monitoring Technologies in Preclinical Drug Discovery and Development. Front Behav Neurosci.
2022 Feb 14;15:758274..
51 fda.gov/drugs/biomarker-qualification-program/biomarker-qualification-program-submission-frequently-asked-questions</p>
<p>34</p>
<p>•  Fit for purpose: intended use of a technology or approach is supported by validation/qualification</p>
<p>information52,53</p>
<p>•  Context of use (COU): defines the manner and purpose of use for a technology or approach (how and
when it will be used).54 This term can generally be applied for any intended use of a methodology.
COU elements include what is measured and in what form, and the purpose of the technology or
approach in the testing of hypotheses or decision making/action.55</p>
<p>52 National Academies of Sciences, Engineering, and Medicine. 2023. Nonhuman Primate Models in Biomedical Research: State of the Science and Future Needs.
Washington, DC: The National Academies Press.
53 FDA and NIH. 2016. BEST (Biomarkers, EndpointS, and other Tools) Resource.
National Academies of Sciences, Engineering, and Medicine. 2023. Nonhuman Primate Models in Biomedical Research: State of the Science and Future Needs.
Washington, DC: The National Academies Press.
54 Baran SW, Brown PC, Baudy AR, Fitzpatrick SC, Frantz C, Fullerton A, Gan J, Hardwick RN, Hillgren KM, Kopec AK, Liras JL, Mendrick DL, Nagao R, Proctor WR,
Ramsden D, Ribeiro AJS, Stresser D, Sung KE, Sura R, Tetsuka K, Tomlinson L, Van Vleet T, Wagoner MP, Wang Q, Arslan SY, Yoder G, Ekert JE. Perspectives on the
evaluation and adoption of complex in vitro models in drug development: Workshop with the FDA and the pharmaceutical industry (IQ MPS Affiliate). ALTEX.
2022;39(2):297–314. doi: 10.14573/altex.2112203. Epub 2022 Jan 21..
55 National Academies of Sciences, Engineering, and Medicine. 2023. Nonhuman Primate Models in Biomedical Research: State of the Science and Future Needs.
Washington, DC: The National Academies Press.</p>
<p>35</p>
<p>APPENDIX B – NIH Request for Information (RFI)
The NIH posted a Request for Information (RFI) for the community to share insights on the challenges and
opportunities for the further development and use of novel alternative methods (NAMs) in research (NOT-OD-
23-140), with the NIH Guide Notice copied below.56 Input received from the RFI was shared with the ACD
Working Group on Catalyzing the Development and Use of Novel Alternative Methods to Advance Biomedical
Research to inform their recommendations and report. The full submissions to the RFI are posted on
osp.od.nih.gov/get-involved/previously-released-compiled-public-comments.</p>
<p>56 grants.nih.gov/grants/guide/notice-files/NOT-OD-23-140.html</p>
<p>36</p>
<p>Request for Information on Catalyzing the Development and Use of
Novel Alternative Methods to Advance Biomedical Research
Notice Number:
NOT-OD-23-140</p>
<p>Key Dates</p>
<p>Release Date:
June 12, 2023</p>
<p>Response Date:
September 5, 2023</p>
<p>Related Announcements
NOT-OD-23-164: Notice to Extend the Response Date of NOT-OD-23-140: Request for Information on Catalyzing the
Development and Use of Novel Alternative Methods to Advance Biomedical Research</p>
<p>Issued by
Office of The Director, National Institutes of Health (OD)</p>
<p>Purpose
The National Institutes of Health (NIH) seeks public input on challenges and opportunities for the further development and
use of novel alternative methods (NAMs) in biomedical research. NIH investment in these methods have proven beneficial
tools across basic and clinical research studies, being developed and applied to interrogate cancer, diabetes,
cardiovascular disease, Alzheimer's disease, infectious disease, rare diseases, and more. Each NAM approach has
unique strengths and limitations that vary depending on the specific research question being addressed.</p>
<p>To identify areas in which the development and use of NAMs provide the most value to biomedical research, NIH sought
the assistance of the Advisory Committee to the Director (ACD), an advisory group that provides advice on matters
pertinent to NIH mission responsibilities in the conduct and support of biomedical research, medical science, and
biomedical communications. The purpose of this request is to inform the NIH and the development of the ACD’s
recommendations on high-priority areas for future investment.</p>
<p>Background</p>
<p>Biomedical researchers rely on a combination of innovative methods, models, and technologies to answer complex
questions about human health and disease. The use of any given approach is based on its ability to answer the research
question under study. While animal models remain an invaluable resource for researchers'  addressing the complexity of
human biology, rapid advances in technology are catalyzing the development and use of complementary, nonanimal
based approaches. These “novel alternative methods” (NAMs) include in chemico strategies (e.g., experiments on
biological molecules like DNA and proteins in test tubes); in vitro methods (e.g., exploring the nature of cells and tissues
by culturing them in sterile chambers); and in silico computational models that simulate how these biological systems work
and predict outcomes to refine hypotheses (e.g., to define how potential drugs interact with their biological targets and to
refine clinical intervention and procedures that increase patient safety and treatment efficacy).  The development of these
NAMs holds tremendous promise for increasing the tools available to achieve the NIH mission and potentially reduce and
refine the future use of animals in some areas of research in the future.</p>
<p>To identify areas in which the development and use of NAMs provide the most value to biomedical research, in January
2023, the NIH Director charged an Advisory Committee to the Director (ACD) Working Group with articulating high-priority
areas for NIH investment (see ACD Working Group on Catalyzing the Development and Use of Novel Alternative Methods
to Advance Biomedical Research - NIH Advisory Committee to the Director). In pursuit of its charge, the ACD Working
Group has been assessing the value and limitation of NAMs and needs for the development of new and/or more effective
NAMs. The ACD Working Group has also been meeting with experts across research sectors to understand both public
and private sector investment and identify gaps/areas of opportunity to maximize the value of NAMs to advance
biomedical research. Collectively, these activities are being summarized in the ACD Working Group’s initial landscape</p>
<p>37</p>
<p>assessment on the challenges and opportunities in the development and use of NAMs that will be used to inform its final
recommendations regarding high-priority areas for NIH investment.</p>
<p>Request for Information</p>
<p>To support the activities of the ACD Working Group, NIH is requesting information from the interested individuals and
communities on challenges and opportunities for the development and use of NAMs in biomedical research. Input sought
includes, but is not limited to, feedback on the following:</p>
<p>•  The use of novel alternative methods to study human biology, circuits, systems, and disease states. The
value of any modeling approach is based on the assertion that known similarities between the model and the
subject matter permit conclusions that additional features observed in the model will also be observed in the domain
to which the model is applied. An optimal future state is one in which our understanding of human biology is
sufficient to design modeling systems that accurately reflect the complexity of that biology. NIH is particularly
interested in hearing how NAMs:</p>
<p>o  are currently being developed and/or used successfully, including features that maximize scientific utility:
o  are advancing progress into understanding specific biological processes or human states, including potential</p>
<p>limitations to addressing human variability; and</p>
<p>o  could be truly revolutionary for understanding/treating human health, including currently underserved areas of</p>
<p>biomedical research.</p>
<p>•  Approaches for catalyzing the development and validation of novel alternative method technologies. Many
of the issues with rigor and translatability in animal models must also be addressed for non-animal models, such as
considerations of human biological relevance, study design, statistical analysis, data sharing, and reporting.
However, there are additional considerations for rigor and translatability that are unique to the development of
NAMs, where development of new technologies and methodologies can outpace scientific consensus on standards.
NIH is particularly interested in hearing from the public on:</p>
<p>o  challenges for building in robustness, replicability, reproducibility and reliability of the technologies and the</p>
<p>ensuing datasets;</p>
<p>o  strategies for bolstering technology readiness and reliability these technologies; and
o</p>
<p>factors potentially limiting the successful integration of these technologies across research approaches and
potential solutions.</p>
<p>•  Strategies for maximizing the research value of novel alternative method technologies. Depending on the
biological system or disease state, different combinations of methods may be required to provide the strongest
body of evidence. NIH is particularly interested in hearing from the public on how to scale these technologies to
more effectively advance scientific inquiry or improve translation, including:
o  areas in which coordinated approaches across research disciplines or research sectors would dramatically</p>
<p>advance the development and or use of these technologies.</p>
<p>o  approaches for sharing technology deployment equitably across labs, including incentives for reliable and</p>
<p>reproducible methods integration.
factors for consideration when maximizing translatability and minimizing bias regarding human variability.</p>
<p>o</p>
<p>How to submit a response</p>
<p>All comments must be submitted electronically at https://osp.od.nih.gov/request-for-information-rfi-catalyzing-the-
development-and-use-of-novel-alternative-methods-to-advance-biomedical-research . It is not necessary to address each
question/item.</p>
<p>Responses must be received by 11:59:59 pm (ET) on September 5, 2023.57</p>
<p>Responses to this RFI are voluntary and may be submitted anonymously. You may also voluntarily include your name and
contact information with your response. Other than your name and contact information, please do not include in the
response any personally identifiable information or any information that you do not wish to make public. Proprietary,
classified, confidential, or sensitive information should not be included in your response. After OSP has finished reviewing
the responses, the responses may be posted to the OSP website without redaction.</p>
<p>57 Submission deadline was originally August 16, 2023 prior to extension.</p>
<p>38</p>
<p>Inquiries
Please direct all inquiries to:</p>
<p>NIH Office of Science Policy</p>
<p>Telephone: 301-496-9838</p>
<p>Email: SciencePolicy@od.nih.gov</p>
<p>39</p>
<p>APPENDIX C – Agenda and Participants, August 2023 Public Meeting</p>
<p>On August 21, the NIH held a public workshop on approaches, challenges, and opportunities relating to the
development of Novel Alternative Methods (NAMs). The workshop also featured discussion on identifying
incentives and barriers to successful implementation of NAMs technologies. The agenda of the meeting is
copied below. A recording of the event can be found at: videocast.nih.gov/watch=49776. A meeting summary
is posted at: www.acd.od.nih.gov/working-groups/novel-alternatives.html.</p>
<p>40</p>
<p>NOVEL ALTERNATIVE METHODS (NAMS) WORKING GROUP
CATAYLZING DEVELOPMENT AND USE OF NOVEL ALTERNATIVE METHODS</p>
<p>August 21, 2023 (all times ET)
(https://videocast.nih.gov/watch=49776)
AGENDA</p>
<p>9:00 AM</p>
<p>Welcome
Howard Chang, MD, PhD &amp; Lyric Jorgenson, PhD – ACD NAMs Working Group Co-Chairs</p>
<p>The Opportunities and Challenges for NAMs in Biomedical Research</p>
<p>9:15 AM
Successful deployment of NAMs, whether for conducting basic research, uncovering disease mechanisms, or
translating knowledge into products or practice, relies on bringing together disciplines, technologies, and data.
This session focuses on research areas for which NAMs have been impactful to identify best practices for
leveraging these approaches.</p>
<p>Moderator: Nancy Lane, MD – ACD NAMs Working Group
Presenters:</p>
<p>•  Nathan Price, PhD – Thorne HealthTech (novel mechanisms)
•  Thomas Hartung, MD – John Hopkins University (translation/product development)
•  Nicole Kleinstreuer, PhD – U.S. National Institutes of Health (regulatory processes)
•  Chirag Patel, PhD – Harvard Medical School (inter-individual differences)</p>
<p>10:30 AM</p>
<p>BREAK</p>
<p>Cross Sector Approaches for Driving NAMs Use and Development</p>
<p>10:45 AM
Each sector within the biomedical research enterprise has a role to play in catalyzing the development and use
of NAMs. This session focuses on the unique and complementary efforts underway to identify synergies and
potential gaps in needed collaboration.</p>
<p>Moderator: Danilo Tagle, PhD – ACD NAMs Working Group
Presenters:</p>
<p>•  Alex Carlisle, PhD – National Alliance Against Disparities in Patient Health (nonprofit)
•  Elijah Peterson, PhD – U.S. National Institute of Standards and Technology (government)
•  Yvette Seger, PhD – Federation of American Societies for Experimental Biology (scientific</p>
<p>society)</p>
<p>12:00 PM</p>
<p>BREAK</p>
<p>41</p>
<p>DISCUSSION OF HIGH PRIORITY NEEDS
The following sessions focus on identifying potential high priority needs for catalyzing NAMs use and
development with human applicability to (1) advance progress into understanding specific biological processes
or states or (2) augment the tools and capabilities for biomedical research to complement and/or potentially
replace traditional models. Specific goals will include identifying incentives for integrating efforts and barriers
to success.</p>
<p>1:00 PM</p>
<p>Developing Integrated and Multi-System Models
Moderator: Szczepan Baran, VMD – ACD NAMs Working Group
Discussants:</p>
<p>•  Graca Almeida-Porada, MD, PhD – Wake Forest University
•  Blanca Rodriguez, PhD – University of Oxford
•  Roser Vento-Tormo, PhD – Wellcome Sanger Institute
•  Terry Van Vleet, PhD – AbbVie</p>
<p>2:00 PM</p>
<p>BREAK</p>
<p>2:15 PM</p>
<p>Leveraging Diverse Datasets for Maximally Useful NAMs</p>
<p>Moderator: Gordana Vunjak-Novakovic, PhD – ACD NAMs Working Group
Discussants:</p>
<p>John Burke, PhD – Applied Biomath</p>
<p>•
•  Anna Gourmelon – Organisation for Economic Co-operation and Development
•  Donna Mendrick, PhD – U.S. Food and Drug Administration
•
•</p>
<p>Ivan Rusyn, MD, PhD – Texas A&amp;M University
James Zou, PhD – Stanford University</p>
<p>3:15 PM</p>
<p>BREAK</p>
<p>3:30 PM</p>
<p>Equitably Deploying Robust and Reliable NAMs into Practice</p>
<p>Moderator: Antonio Baines, PhD – ACD NAMs Working Group
Discussants:</p>
<p>Jessie Carder – U.S. Department of Agriculture</p>
<p>•
•  Megan LaFollette, PhD – The North American 3Rs Collaborative
•  Michael Moore, PhD – Tulane University; AxoSim
•  Manu Platt, PhD – U.S. National Institutes of Health
•  Nicholas Tatonetti, PhD – Columbia University</p>
<p>4:30 PM</p>
<p>DISCUSSION AND NEXT STEPS
Howard Chang, MD, PhD &amp; Lyric Jorgenson, PhD – ACD NAMs Working Group Co-Chairs</p>
<p>5:00 PM</p>
<p>ADJOURN</p>
<p>42</p>












                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "../..", "features": ["content.tabs.link"], "search": "../../assets/javascripts/workers/search.d50fe291.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../assets/javascripts/bundle.50899def.min.js"></script>
      
    
  </body>
</html>